<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<head>

<meta charset="utf-8" />
<meta name="generator" content="quarto-1.6.42" />

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />

<meta name="author" content="Raahil Madhok" />
<meta name="author" content="Matt Braaksma" />
<meta name="author" content="Ryan McWay" />
<meta name="author" content="Jovin Lasway" />

<title>The Economic Value of Biodiversity in India</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>

<!-- htmldependencies:E3FAD763 -->

  
  <script>
    document.addEventListener("DOMContentLoaded", function () {
      var head = document.getElementsByTagName("head")[0];
      var link = document.createElement("link");
      link.rel = "stylesheet";
      link.href = "https://cdn.jsdelivr.net/npm/katex@0.13.11/dist/katex.min.css";
      head.appendChild(link);

      var script = document.createElement("script");
      script.type = "text/javascript";
      script.src  = "https://cdn.jsdelivr.net/npm/katex@0.13.11/dist/katex.min.js";
      script.async = false;
      script.addEventListener('load', function() {
        var mathElements = document.getElementsByClassName("math");
          var macros = [];
          for (var i = 0; i < mathElements.length; i++) {
            var texText = mathElements[i].firstChild;
            if (mathElements[i].tagName == "SPAN") {
              window.katex.render(texText.data, mathElements[i], {
                displayMode: mathElements[i].classList.contains('display'),
                throwOnError: false,
                macros: macros,
                fleqn: false
              });
            }
          }
      });
      head.appendChild(script);
    });
  </script>
  

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body>

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <div id="quarto-toc-target"></div>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">The Economic Value of Biodiversity in India</h1>
<p class="subtitle lead">Running Notes</p>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Authors</div>
    <div class="quarto-title-meta-contents">
             <p>Raahil Madhok </p>
             <p>Matt Braaksma </p>
             <p>Ryan McWay </p>
             <p>Jovin Lasway </p>
          </div>
  </div>
    
  
    
  </div>
  


</header>

<nav id="TOC" role="doc-toc">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#project-description" id="toc-project-description">Project Description</a></li>
  <li><a href="#notes" id="toc-notes">Notes</a></li>
  </ul>
</nav>
<section id="project-description" class="level2">
<h2>Project Description</h2>
<p>This project aims to produce the first measure of willingness to pay (WTP) for biodiversity. This provides a monetary measure to value biodiversity which can be applied to a wide variety of applications in environmental economics and natural captial accounting. The data relies on E-bird observations of diverse bird species observations. The sample is limited to local, long-time users who are residents (not tourists) in India. The methodology to calculate WTP relies on a revealed preference throughh the random utility model (RUM). The RUM determines the value of seeking out diverse species of birds by comparing the value of alternative counterfactual bird siting locations and the cost to go to these counterfactuals. Monetary value is determined through the RUM by estimating the travel cost of the reveal preference compared to counterfactual locations.</p>
</section>
<section id="notes" class="level2">
<h2>Notes</h2>
<p>For notes, we present the most recent first so that notes are chronologically most recent.</p>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-1-contents" aria-controls="callout-1" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
9/12/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-1" class="callout-1-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Raahil, Matt, Ryan, Jovin</p>
</blockquote>
<section id="todo" class="level1">
<h1>TODO:</h1>
<ul>
<li><strong>Matt</strong>: Diagnositics for missing species richness</li>
<li><strong>Matt/Jovin</strong>: Migration component for Shift-share</li>
<li><strong>Ryan/Jovin</strong>: Computing Shift-Share</li>
<li><strong>All</strong>: Review new working paper (has new IV)</li>
</ul>
<section id="agenda" class="level2">
<h2>Agenda</h2>
<ol type="1">
<li>Missing information for species richness by season</li>
</ol>
<ul>
<li>Core issue coming from the imputation with BirdLife data leading to negative coefficient</li>
<li>Several hotspot clusters are missing seasonal information
<ul>
<li>Means that noone is visiting these hotspots previously (e.g., rarely visited)</li>
</ul></li>
<li>Diagnostic: Number of trips per hotspot (confirm they are hotspots)
<ul>
<li>Number of distinct time periods with observations for each hotspot</li>
</ul></li>
</ul>
<ol start="2" type="1">
<li>Migration for Shift-share</li>
</ol>
<ul>
<li>Classify birds are migratory or not</li>
<li>Remove species with missing migratory classification</li>
<li>Keep migratory species to determine migration connections to regions</li>
<li>Jovin has successfully matched a lot of these (fuzzy matching)</li>
</ul>
</section>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-2-contents" aria-controls="callout-2" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
8/27/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-2" class="callout-2-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Ryan, Jovin</p>
</blockquote>
<section id="todo-1" class="level1">
<h1>TODO:</h1>
<ul>
<li><strong>Matt</strong>: We need to meet with you to discuss working on this with MSI</li>
<li><strong>Jovin</strong>: Responsible for Shift</li>
<li><strong>Ryan</strong>: Responsible for Share</li>
</ul>
<section id="plan-to-estimate-shift-share" class="level2">
<h2>Plan to Estimate Shift-Share</h2>
<ol type="1">
<li>Merge the Species range data with the migratory species data
<ul>
<li>Species range data: base-data/species/BOTW_2024_2.gpkg</li>
<li>Migratory species data: base-data/species/species_list_categorized.csv</li>
<li>Merge by: scientific_name</li>
<li>Result: species level data with range map and classifier for migratory</li>
</ul></li>
<li>For each country j, calculate <span class="math inline">\gamma_j</span></li>
</ol>
<p><span class="math display">
\gamma_j = \sum_{s =\text{total num of species in j}}^S \frac{\text{species range overlap India}_{sj}}{S_j}
</span></p>
<ul>
<li>Count the number of species <span class="math inline">s</span> in <span class="math inline">j</span>
<ul>
<li>Species range data: base-data/species/BOTW_2024_2.gpkg</li>
<li>India Borders: data/shp/district-2011</li>
<li>Need to make one shape (spatial union) so any intersection with India shape</li>
<li>Indicator for each species if any overlap of species with India borders</li>
</ul></li>
<li>Spatial overlap of species <span class="math inline">s</span> range over the country border of India. This will create some fraction.
<ul>
<li>Calculate sqkm for India shapefile one time (part 1)</li>
<li>Spatial inner join of range <span class="math inline">s</span> on India.</li>
<li>Calculate sqkm for the inner join (part 2)</li>
<li>Divide part 2 by part 1 to get the fraction of overlap for species range over India borders</li>
</ul></li>
<li>Combine to make <span class="math inline">\gamma_j</span> using the weighted sum <span class="math inline">\sum_s</span>
<ul>
<li>Spatial intersection of species range <span class="math inline">s</span> with countries in the flu dataset</li>
<li>Create the weight <span class="math inline">S_j</span> by counting the number of indicators for intersecting ranges with <span class="math inline">j</span></li>
<li>Create <span class="math inline">\gamma_j</span> for each <span class="math inline">j</span> by summing fractions in (2) divided by weight <span class="math inline">S_j</span></li>
<li>Country <span class="math inline">j</span> Shapefiles: TODO.</li>
</ul></li>
</ul>
<ol type="1">
<li><p>Create the shift by merging <span class="math inline">\gamma_j</span> with the flu data - Flu Data: data/clean/flu-data.csv - Result is time series by t <span class="math display">
Shift_t = \sum_{j = country}^J flu_{jt} \times \gamma_j
</span></p></li>
<li><p>Create the share for each hotspot t</p></li>
</ol>
<p><span class="math display">
Share_d =  \sum_s \frac{\text{Indicator for migratory}_{sd}}{S_d}
</span></p>
<pre><code>1. Start with e-bird data. For each hotspot $d$ get the number of unique species observed in the data
    - e-bird data: Ask Matt what is the best way to do this given the current workflow. 
2. Merge in the migratory data on these species for each hotspot $d$
    - Migratory species data: base-data/species/species_list_categorized.csv
    - Merge by: scientific_name
3. Create the share variable as weighted sum for hotspot $d$
    - For hotspot $d$, count indicator for migratory. 
    - For hotspot $d$, get count of unique species $S_d$
    - Then construct sum of the fractions. </code></pre>
<ol start="5" type="1">
<li>Create the IV by multiplying the shift by the share for each hotspot d</li>
</ol>
<p><span class="math display">
IV_{dt} = shift_t \times share_d
</span></p>
</section>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-3-contents" aria-controls="callout-3" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
8/15/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-3" class="callout-3-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Ryan, Matt, Raahil, Jovin</p>
</blockquote>
<section id="todo-2" class="level1">
<h1>TODO:</h1>
<ul>
<li><strong>Matt</strong>: Demean data for State x month fixed effect and user fixed effects</li>
<li><strong>Matt</strong>: Resolve imputing species richness by estimating correlation of e-bird to bird life data and deflate value
<ul>
<li><strong>Matt</strong>: Degree of Imputation: Summary stats on % of missing in observed and counterfactual data</li>
</ul></li>
<li><strong>Jovin</strong>: Constructing the shift-share IV</li>
</ul>
<section id="agenda-1" class="level2">
<h2>Agenda</h2>
<ol type="1">
<li>Estimation of WTP
<ul>
<li>negative values came from mismatched polygons
<ul>
<li>Difference from the clipped calipers instead of the original simple circles</li>
</ul></li>
<li>weeks sometimes were only 53 (not 54). Solved with iso-week formating that allows for more merging of site attributes</li>
</ul></li>
<li>Data issue
<ul>
<li>we don’t observe species richness</li>
<li>So we are imputing with range maps</li>
<li>When there is missing, we currently fill with species map</li>
<li>Possible species ranges is always higher than recorded value from e-birders</li>
<li>So the imputed values always are super high when missing, and then this makes our estimates really weird.
<ul>
<li>This penalizes the choice. Because counterfactual choice has huge species richness</li>
</ul></li>
<li>Potential Solution: replace missing with historical average (or minimum average) for each district or state in the same time period.
<ul>
<li>Impute that for hotspots with missing data</li>
</ul></li>
<li>Potential Solution: correlation of bird life data and e-bird observations
<ul>
<li>Then we can just deflate imputed value that rescales this to the e-bird values</li>
</ul></li>
<li>What is the main source of missing value?
<ul>
<li>want summary stats on % of missing in observed and counterfactual data</li>
<li>Could make this seasonal species richness for imputed value</li>
</ul></li>
</ul></li>
<li>Fixed Effects Issue
<ul>
<li>NA for standard errors when adding individual fixed effects</li>
<li>Running with python on MSI</li>
<li>Could attempt to manually demean data (user fixed effects) before running</li>
<li>State x month fixed effect and user fixed effects
<ul>
<li>Robustness by district x month fixed effects</li>
<li>This will just be more intense</li>
</ul></li>
<li>mlogit does not allow for fixed effects</li>
<li>The key is to demean within each group
<ul>
<li>hdfe is package in Stata</li>
</ul></li>
</ul></li>
<li>Instrumental Variables
<ul>
<li>Have a map data for bird migration routes
<ul>
<li>Only found 2 species public for India</li>
<li>There are other species we can request access to</li>
</ul></li>
<li>Data avaliable for avian flu outbreak
<ul>
<li>H5N1</li>
<li>2005 - 2025</li>
</ul></li>
<li>Combining the two data sets we can assign outbreaks to specific species that migrate to India</li>
<li>Human cases of avian flu deaths should be controlled for in IV estimation</li>
<li>If we have a shift-share IV, do we need to know the flyways?
<ul>
<li>If this is the only channel, then we don’t need to actually need to know the origin of the bird</li>
</ul></li>
</ul></li>
</ol>
<p><span class="math display">
IV_{dt} = shift_t \times share_d
</span></p>
<p><span class="math display">
IV_{dt} = \text{Wild Avian Flu}_t \times \text{% of species pool from migration}_d
</span></p>
<p><span class="math display">
Shift_t = \sum_{j = country}^J flu_{jt} \times \gamma_j
</span></p>
<p><span class="math display">
\gamma_j = \sum_{s =\text{total num of species in j}}^S \frac{\text{species range overlap India}_{sj}}{S_j}
</span></p>
<ul>
<li>Assumes a uniform distribution of species across their species range
<ul>
<li>This is an improvement over distance from India as the weight</li>
</ul></li>
</ul>
<p><span class="math display">
Share_d =  \sum_s \frac{\text{Indicator for migratory}_{sd}}{S_d}
</span></p>
</section>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-4-contents" aria-controls="callout-4" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
8/1/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-4" class="callout-4-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Ryan, Matt, Raahil</p>
</blockquote>
<section id="todo-3" class="level1">
<h1>TODO:</h1>
<ul>
<li><strong>Matt</strong>: Re-estimating WTP using new sample of users (expanded)</li>
<li><strong>Matt</strong>: Adding a Visual: Attempting to run the simple regression of biodiversity and hotspot visitations</li>
<li><strong>Jovin</strong>: Complete the IV design by finding accompanying data. Maps of migration patterns. Papers to cite for this phenomenon.</li>
<li><strong>Raahil/Ryan</strong>: Write up the model in LaTeX</li>
</ul>
<section id="agenda-2" class="level2">
<h2>Agenda</h2>
<ol type="1">
<li>Figures describing variation
<ul>
<li>Ryan presented some post estimation figures we tabled for later</li>
<li>Matt presented some maps showing variation over space</li>
<li>Raahil suggested we produce a margins plots. We run an endogenous TWFE model of species richness on number of visitors. Then we plot the marginal effects at different levels of biodiversity.</li>
</ul></li>
<li>First estimates of WTP using super users
<ul>
<li>Matt is able to run the conditional logit model. Not able to run the mixed logit model</li>
<li>Our <span class="math inline">\alpha</span> for price has the wrong sign. This makes biodiversity WTP a negative value</li>
<li>We think that the sample may be too restrictive resulting in weird results</li>
<li>Additionally, Matt is working through some data issues in the counterfactual data set that is dropping 1.1% of site attributes</li>
<li>And there is colinearity amongst site attributes when running the conditional logit, so they are currently dropped in the model run.</li>
</ul></li>
<li>Modeling updates using IV method
<ul>
<li>Ryan has a clear idea of how to model the IV into our mixed logit model</li>
<li>The key is to rely on Jacob Bradt’s model using control functions.</li>
<li>We estimate the first stage linear regression. Residualize. And use the residuals in our discrete choice method (mixed logit).</li>
<li>We will need to bootstrap the 2nd stage in our to recover standard errors.</li>
</ul></li>
<li>Bird Migration Instruments
<ul>
<li><ol type="i">
<li>Migration patterns: From the literature I’ve compiled so far:</li>
</ol>
<ul>
<li>Peak arrivals: August - November</li>
<li>Peak departures: February - April</li>
<li>Low migration periods: May - July, and December - January</li>
</ul></li>
<li>The Inbound migration: Birds arrive in India primarily from the northern breeding grounds, especially in Mongolia, Kazakhstan, and Siberia, during the August-November window to overwinter in wetlands and coastal areas.</li>
<li>The Outbound migration: Birds depart February-April, returning to their breeding sites for nesting.</li>
<li><ol start="2" type="i">
<li>Drivers of bird migration and IV framework: Ecology literature indicates that migration along the Central Asian Flyway is mainly driven by (i) Ecological reasons, india’s wetlands, lakes, and agricultural landscapes provide abundant food and shelter during winter, (ii) Climatic reasons: Wrmer winter conditions compared to the northern latitudes reduce suruval stress (iii) Stopover importance: Key sites such as Chilika, Keoladeo, Pong Dam, etc serve as staging areas for fueling before onward flights.</li>
</ol></li>
<li>Recent literature uses environmental shocks as instruments (Noack et al., 2024; Meng et al, 2025). For India’s bird migration context, two IV strategies appear promising.
<ul>
<li>Weather shock IV: Baseline district exposure (from our eBird) interacting with NVDI and weather anomalies in the breeding grounds.</li>
<li>Disease shock IV: I find this is more interesting because a recent work by Yang et al., 2024 at Nature Communications finds avian influenza outbreaks do not trigger migration but do affect survival and presence of birds at the Indian stopovers to a very large extent. This makes it a credible instrument because: (i) outbreaks are exogenous to the local economic variables and (ii) outbreak timing is tied to a peak of migration window/CAF (Aug-Nov, Feb-Apr) in India, allowing shocks to coincide with relevant bird presence.</li>
</ul></li>
<li><ol start="3" type="i">
<li>Data collection: Potential outbreak data sources, shapefiles include (i) FAO EMPRES-i+; (ii) WOAH/WAHIS, which tracks the outbreak locations, date, disease type, host species, and status. Lastly, GISAID shapefile data, which is an avian influenza sequence database, tracks the outbreak locations (used in Elbe &amp; Buckland-Merrett, 2017, Global Challenges Journal; Yang et al, 2024, Nature Communications).</li>
</ol></li>
<li><ol start="4" type="i">
<li>Next step: Performing a spatial join between outbreak data and our ebird-based hotspots, aggregating outbreak counts to the district/hotspot-year panel level. I think this will allow us to construct the avian influenza IV as = Baseline exposure in district/hotspot d x outbreak shock at time t.</li>
</ol></li>
</ul></li>
</ol>
</section>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-5-contents" aria-controls="callout-5" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
6/2/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-5" class="callout-5-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Ryan, Matt, Jovin, Raahil</p>
</blockquote>
<section id="todo-4" class="level1">
<h1>TODO:</h1>
<ul>
<li><strong>Matt</strong>: Finish species richness site attribute (cluster-time level)</li>
<li><strong>Matt</strong>: Finish computing congestion site attribute</li>
<li><strong>Matt</strong>: Check why sample size so big. Check number of elements in the choice set for validation.</li>
<li><strong>Matt/Ryan</strong>: Compute species richness in radius <em>around home</em> (varying at same time unit as other species richness attribute)</li>
<li><strong>Jovin</strong>: Fix travel cost. I suggest checking it over with Matt/Ryan</li>
<li><strong>Raahil/Ryan</strong>: Work through paper framing</li>
</ul>
<section id="agenda-3" class="level2">
<h2>Agenda</h2>
<ol type="1">
<li>Species Richness</li>
</ol>
<ul>
<li>Maps for imputation can’t have the shannon index. Because we don’t have abundance counts</li>
<li>But we can do an index for the e-bird data hotspots</li>
<li>Estimates of bird life and expected species richness</li>
<li>Monthly values or seasonal values</li>
</ul>
<ol start="2" type="1">
<li>Other Site Attributes</li>
</ol>
<ul>
<li>Have one site data set for temp, rain, and tree cover</li>
<li>This is at the hotspot cluster level</li>
</ul>
<ol start="3" type="1">
<li>Congestion</li>
</ol>
<ul>
<li>Ready to go code, just not run yet</li>
<li># of users at hotspot cluster monthly</li>
</ul>
<ol start="4" type="1">
<li>Travel Cost</li>
</ol>
<ul>
<li>Jovin has choice set and code that estmates this for a first attempt</li>
<li>Matched to GDP data to impute wages</li>
<li>Estimate Euclidean distance between households and hotspot clusters</li>
<li>Final data for RUM should be choice set dimensions (for our checks)</li>
</ul>
<ol start="5" type="1">
<li>Measure of Experience at the Individual Level</li>
</ol>
<ul>
<li>Counts of trips (counter)</li>
<li>Count for # of different hotspots (counter)</li>
<li>So ratio tells us if you have preference for location (“Habit Formation”).
<ul>
<li>Low ratio is little location preference</li>
<li>High ratio is high location preference</li>
</ul></li>
</ul>
<ol start="6" type="1">
<li>Conceptual</li>
</ol>
<ul>
<li>Birds vs. non-bird still valued as travel cost. So both are still measures of nature recreation.</li>
<li>Monetizing value &gt;&gt; Add in contribute to GEP or ecosystem service value literature.</li>
<li>Novelty: Why is WTP lower in Developing Countries? Welfare for environmental quality in envirodevonomics framework (Env-Dev framing)</li>
</ul>
</section>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-6-contents" aria-controls="callout-6" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
5/16/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-6" class="callout-6-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Ryan, Matt, Jovin, Raahil</p>
</blockquote>
<section id="todo-5" class="level1">
<h1>TODO:</h1>
<ul>
<li>Seperate meetings for Matt/Jovin (coding) and Ryan/Raahil (conception) for our specializations</li>
</ul>
<section id="agenda-4" class="level2">
<h2>Agenda</h2>
<ol type="1">
<li>Finalize the site attribute data and create the master dataset</li>
</ol>
<ul>
<li>Matt and Ryan – complete by next meeting (May 16)</li>
<li>We have monthly temperature and rainfall</li>
<li>We have annual tree cover data</li>
<li>These are as zonal statistics (average for 10KM radius of hotspot)
<ul>
<li>In E-bird, we have the average distance walked around hotspot</li>
<li>Or do we avoid this work because temp, rain, and tree cover does not vary much at such a small spatial scale.</li>
<li>Temp, rainfall, and tree are at higher resolutions. So as long as the radius is less than the input data this will not vary.</li>
</ul></li>
<li>We need to think more about mapping of hotspots and hotspot clusters.</li>
<li>Need to add a fourth site attribute: congestion
<ul>
<li>How frequently other users are using this site?</li>
<li>Aggregate number of user trips for each hotspot but will need to do this as a lag (week before)</li>
<li>So this is expected congestion.</li>
</ul></li>
<li>We need to transform this into a panel of counterfactual sets appended to the observed trips.
<ul>
<li>Not balanced across counterfactual trips and time</li>
</ul></li>
</ul>
<ol start="2" type="1">
<li>Calculate travel costs – see notes for guidance; this will be added to the site attribute script</li>
</ol>
<ul>
<li>For GDP per capita data, use gridded data from here and extract the value closest to each users’ home. Then use the formula from Kolstoe/cameron or lloyd-smith to calculate travel costs.</li>
<li>Jovin – complete by next meeting (May 16)</li>
<li>Collected 2014-2024 GDP per capita data</li>
<li>Laptop trouble estimating, working with MPC.</li>
<li>Could run the travel cost with a small random sample to make this managable for Jovin’s laptop capacity.</li>
<li>Can re-run the pipeline so we are running for only frequent users (1.6M instead of the 3M)</li>
</ul>
<ol start="3" type="1">
<li>Species richness</li>
</ol>
<ul>
<li>Matt and Ryan – bring three ideas for next meeting (May 16)</li>
<li>Close to being done (Matt taking the lead)</li>
<li>Taking raw e-bird data and getting monthly counts</li>
<li>Need to translate from hotspots to hotspot clusters</li>
<li>Species ranges (for missing) shapefiles are crashing the code.</li>
<li>All of this is to get a species at a hotspot so we can turn this into a richness index
<ul>
<li>Will want this merged as richness for same month/week from previous year</li>
<li>In the app, you can see in real-time species count from their favorite places.</li>
<li>So likely quite responsive. Last week is best</li>
<li>Can also do monthly to replicate results from month last year from Kolsto paper method.</li>
</ul></li>
<li>We aim to use the simpson and shanon index from <code>vegan</code> package</li>
</ul>
<ol start="4" type="1">
<li>Individual Attributes</li>
</ol>
<ul>
<li>With individual fixed effects, we partial out any time-invariant</li>
<li>From the e-bird data:
<ul>
<li>How frequently you went in the past year</li>
<li>Average distance traveled in the past year</li>
<li>Is there any measure of learning over time? Value of recreation is not static
<ul>
<li>Experience index</li>
<li>Cumulative experience (simple counter of trips up unti that point)</li>
<li>Could weight this by how much you walk, and taxonomy of bird level data (raw text file of trip-species level)</li>
<li>Could collapse species count to trip level.</li>
</ul></li>
</ul></li>
</ul>
</section>
<section id="project-management" class="level2">
<h2>Project Management</h2>
<ul>
<li>Meetings need to be in person.</li>
<li>Jovin update meeting calendar</li>
<li>Start assigning tasks instead of asking for volunteers</li>
<li>Need to come up with a specialization of tasks.</li>
<li>Coding and Data: Matt (lead), Jovin, Ryan (assist)</li>
<li>Conceptualize: Ryan (lead), and Raahil
<ul>
<li>How our model departs from workhouse.</li>
<li>Literature review for our novelty</li>
</ul></li>
<li>Some tasks:
<ul>
<li>Google maps driving time</li>
<li>inaturalist species richness mapping</li>
</ul></li>
</ul>
</section>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-7-contents" aria-controls="callout-7" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
5/15/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-7" class="callout-7-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Ryan, Matt</p>
</blockquote>
<section id="coding-notes" class="level2">
<h2>Coding Notes</h2>
<ul>
<li>Ryan is taking lead on estimating travel cost.</li>
<li>We need a seperate hotspots shapefile for running zonal stats</li>
<li>We need unique hotspot site IDs and hotspot cluster IDs</li>
<li>Explor alternatives to making hotspot clusters</li>
<li>Name for the master data</li>
<li>Need to add attributes to hotspot clusters (script 4)</li>
<li>Mapping of hotspot cluster ID to hotspot IDs</li>
</ul>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-8-contents" aria-controls="callout-8" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
5/2/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-8" class="callout-8-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Raahil, Matt</p>
</blockquote>
<section id="agenda-5" class="level2">
<h2>Agenda</h2>
<ol type="1">
<li>Site attributes data progress
<ul>
<li>In progress</li>
</ul></li>
<li>Individual attributes data progress</li>
<li>Coding sample selection progress
<ul>
<li>Done, flexible to time period and years (e.g. quarterly for at least 8 years)</li>
</ul></li>
</ol>
</section>
<section id="site-attributes" class="level2">
<h2>Site Attributes</h2>
<ul>
<li>Still gathering geospatial data
<ul>
<li><a href="https://lpdaac.usgs.gov/products/mod44bv061/">MODIS VCF forest cover</a></li>
<li><a href="https://cds.climate.copernicus.eu/datasets/reanalysis-era5-land?tab=download">Copernicus ERA5-Land</a>
<ul>
<li><a href="https://bluegreen-labs.github.io/ecmwfr/">ecmwfr</a>, R API interface</li>
</ul></li>
</ul></li>
<li>Run zonal statistics on buffer around hotspots</li>
<li>If data becomes to large to easily share, we can start using Globus and MSI’s tier 2 data storage</li>
</ul>
</section>
<section id="next-tasks" class="level2">
<h2>Next tasks</h2>
<ol type="1">
<li>Merge site and individual attributes to make master dataset</li>
<li>Calculate travel cost
<ul>
<li>Start with linear distance for now</li>
<li>Calculate cost
<ul>
<li>Refer to Lloyd-Smith and Cameron/Kolstoe</li>
<li>Travel cost = 2 (round trip) x distance x 1/3 x wage (from GDP)</li>
</ul></li>
</ul></li>
<li>Species Richness
<ul>
<li>Decide on method:
<ul>
<li>Weitzman index</li>
<li>iNaturalist</li>
<li>Simpson index</li>
<li>Shannon index</li>
</ul></li>
</ul></li>
<li>Looking ahead
<ul>
<li>Look at code for estimating mixed logit with big data
<ul>
<li><a href="https://github.com/joemolloy/fast-mixed-mnl">mixl</a>, C++ based R package</li>
<li><a href="https://xlogit.readthedocs.io/en/latest/index.html">xlogit</a>, GPU-Accelerated Python Package</li>
</ul></li>
<li>How to properly weight the WTP for aggregation <!-- - [Weighting the Evidence: A Rank-Dependent Model of Outdoor Recreation](https://doi-org.ezp1.lib.umn.edu/10.1086/732185) --></li>
<li>Research how to include fixed effects in utility functions (alternative specific constant)
<ul>
<li>User fixed effects</li>
<li>hotspot fixed effects</li>
</ul></li>
</ul></li>
</ol>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled" data-collapsse="true">
<div class="callout-header d-flex align-content-center">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
4/25/2025
</div>
</div>
<div class="callout-body-container callout-body">
<blockquote>
<p>Matt, Ryan</p>
</blockquote>
<section id="tasks" class="level2">
<h2>Tasks:</h2>
<ul>
<li>Matt: Download data and share</li>
<li>Ryan: Start writing pseudocode in the new scripts.</li>
<li>Both: Play with ClimateR</li>
</ul>
</section>
<section id="site-attributes-1" class="level2">
<h2>Site Attributes</h2>
<ul>
<li>Limitation:
<ul>
<li>It takes time to build GIS data. So a lot of the attributes are not updated for 2024/25</li>
<li>ClimateR</li>
</ul></li>
<li>Division of the attributes to collect
<ul>
<li>Protected area (binary or distance) – fuzzy borders leads to spillovers
<ul>
<li>Matt</li>
<li>WDPA (https://www.protectedplanet.net/en/thematic-areas/wdpa?tab=WDPA)</li>
</ul></li>
<li>Tree cover – higher willingess to pay for forest
<ul>
<li>Matt</li>
<li>https://data.fs.usda.gov/geodata/rastergateway/treecanopycover/</li>
<li>https://data.globalforestwatch.org/</li>
<li>https://glad.umd.edu/dataset/global-2010-tree-cover-30-m</li>
<li>Worse comes to worse we use the ESA data</li>
<li>Or we could use NDVI data as a proxy.</li>
</ul></li>
<li>Rainfall (monthly) - seasonality
<ul>
<li>Ryan</li>
<li>https://ftp.cpc.ncep.noaa.gov/GIS/GRADS_GIS/GeoTIFF/GLB_DLY_PREC/</li>
<li>CHRIPS SPI data</li>
<li>https://mikejohnson51.github.io/climateR/index.html</li>
</ul></li>
<li>Temperature (monthly) - seasonality
<ul>
<li>Ryan</li>
<li>GISS Surface Temperature Analysis (GISTEMP v3)</li>
<li>Surface air temperature (no ocean data), 250km smoothing (9 MB)</li>
<li>Might need the land mask</li>
<li>https://data.giss.nasa.gov/gistemp/</li>
</ul></li>
<li>Distance to the coast – species gradient to the coast
<ul>
<li>Ryan</li>
<li>Dissolve the districts data.</li>
<li>Make a polyline and measure distance to the nearest line</li>
</ul></li>
<li>Species richness from the previous week (expectation) – and any other measures of biodiversity
<ul>
<li>Both Matt and Ryan</li>
<li>Weitzmen index using ebird data to generate diversity score</li>
<li>https://scholar.harvard.edu/files/weitzman/files/on_diversity.pdf</li>
<li>Validate with inaturalist data on birds for india</li>
<li>Could test a couple of different indices</li>
</ul></li>
</ul></li>
<li>How we are sharing the site attribute data
<ul>
<li>Matt will have the GDrive link.</li>
</ul></li>
<li>Updating the config files
<ul>
<li>Once we have data download</li>
</ul></li>
<li>Start script to merge site attributes to sites
<ul>
<li>What to do about 4.demographics. Seems like that should be site attributes.</li>
<li>Noteably, some of the data will need to become panel.
<ul>
<li>Rainfall</li>
<li>Temperature</li>
<li>Protected Areas</li>
<li>Tree cover</li>
<li>species richness (we need to dig some that has a time dimension)</li>
</ul></li>
<li>Static:
<ul>
<li>Distance to coast</li>
</ul></li>
</ul></li>
</ul>
</section>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-10-contents" aria-controls="callout-10" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
4/18/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-10" class="callout-10-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Raahil, Matt, Jovin, Ryan</p>
</blockquote>
<section id="sample-selection-updates" class="level2">
<h2>Sample Selection Updates</h2>
<ul>
<li>different biodiversity levels by incorporating interaction terms in locals and tourists. WTP for nature tourism vs WTP for nature recreation or biodiverstiy
<ul>
<li>If we estimate both, then we can compare both measures. We can also place it within the larger literature of many WTP for both subsets of ecosystem services and the population using those services.</li>
<li>Broadly compare conservation, nature recreation, tourism, or biodiversity</li>
<li>We need to frame this as a development paper as the lambda in the Envirodevonomics paper</li>
</ul></li>
<li>Frequency of observations are consistent over time and potentially visits near protected areas.
<ul>
<li>Need to capture seasonanlity (time fixed effects)</li>
<li>Need to consider some aspect of congestion (can calculate that as a site attribute)</li>
<li>If you go two months a year you are less likely to be a tourist</li>
</ul></li>
</ul>
</section>
<section id="literature-review-updates" class="level2">
<h2>Literature Review Updates</h2>
<ul>
<li>We can uses species richness as an explanatory variable in the model. But could add other biodiversity measures beyond bird diversity.</li>
<li>Need to think about how to frame this in the developing context instead of the environment context</li>
<li>Very few WTP for environmental quality in development economics space</li>
<li>Would be great if our WTP measure could be placed within a welfare model so we can make counterfactual statements
<ul>
<li>Need to have homogenous agents to aggregate for welfare gains &gt;&gt; e-bird users are not representative of population</li>
<li>Could overcome this by population demographic weights to make e-bird users representative</li>
<li>But welfare statements are a big improvement over the other e-bird papers</li>
</ul></li>
<li>Could explore the WTP measure of space and time. So we go beyond the cross-sectional estimates in the literature
<ul>
<li>If we have 1st and 2nd moment, then we can assign individuals an individual WTP.</li>
</ul></li>
</ul>
</section>
<section id="grant-proposal" class="level2">
<h2>Grant Proposal</h2>
<ul>
<li>Jovin will provide an update.</li>
</ul>
</section>
<section id="initial-results-by-raahil" class="level2">
<h2>Initial Results by Raahil</h2>
<ul>
<li>We are getting coverage of the country</li>
<li>Some overlap of population density and users &gt;&gt; likely representative</li>
<li>Most hotspots and users are in the south (consistent with e-birder users)</li>
<li>Hotspots data could be weighted by visits</li>
<li>Should plot protected areas overlapping hotspots</li>
<li>Demographic t-test shows e-bird users are richer on average. But we can use this to back out our inverse weight for the welfare calculations
<ul>
<li>E-bird users are selecting into this where there are richer places (so can afford this hobby)</li>
</ul></li>
</ul>
</section>
<section id="tasks-1" class="level2">
<h2>Tasks</h2>
<ul>
<li>How do we distinguish ourselves from the Pacific Northwest paper Kolestet 2017</li>
<li>First pass of the sample: users who use this at least 2 months out of the year</li>
<li>Make two new scripts:
<ul>
<li>Merging in site attributes (Can merge in satilette data – radius around the centriod and take the mean – and district data about the site attribtues – merge on district ID)
<ul>
<li>Protected area (binary or distance) – fuzzy borders leads to spillovers</li>
<li>Tree cover – higher willingess to pay for forest</li>
<li>Rainfall (monthly) - seasonality</li>
<li>Temperature (monthly) - seasonality</li>
<li>Distance to the coast – species gradient to the coast</li>
<li>Species richness from the previous week (expectation) – and any other measures of biodiversity</li>
</ul></li>
<li>Estimating the RUM via the mixed multinominal logit</li>
</ul></li>
<li>Email tag for grant proposals</li>
<li>Make a script into a single PDF</li>
</ul>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-11-contents" aria-controls="callout-11" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
3/31/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-11" class="callout-11-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Raahil, Matt, Jovin</p>
</blockquote>
<section id="sample-selection-discussion" class="level2">
<h2>Sample Selection Discussion</h2>
<ul>
<li>We want to ensure that the sample of ebird observers is representative
<ul>
<li>Remove tourists</li>
<li>Remove infrequent users</li>
</ul></li>
<li>Potential strategies
<ul>
<li>Limit to users that log at least a certain number of times per month</li>
<li>Exclude users that post outside of protected areas</li>
<li>Exclude users that include tourism tags in text variable(e.g. “ecolodge”)</li>
</ul></li>
</ul>
</section>
<section id="tasks-2" class="level2">
<h2>Tasks</h2>
<ul>
<li>Matt/Jovin: descriptive analysis on data to determine distributions of frequent users
<ul>
<li>Complete by 4/18</li>
</ul></li>
</ul>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-12-contents" aria-controls="callout-12" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
3/21/2025
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-12" class="callout-12-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Raahil, Matt, Ryan, Jovin</p>
</blockquote>
<section id="action-items" class="level2">
<h2>Action Items:</h2>
<ul>
<li>Everyone confirms that we can run the first 3 basic scripts</li>
<li>Starting the annotated bibliography with 10 most important papers:
<ul>
<li>WTP for envrionmental quality</li>
<li>WTP in developing countries using travel cost</li>
<li>Review of mixed logit methodology</li>
</ul></li>
<li>Next week we need to talk about sample selection in depth next week</li>
</ul>
</section>
<section id="division-of-tasks" class="level1">
<h1>Division of Tasks:</h1>
<ul>
<li><p>Review mixed logit method</p></li>
<li><p>(Matt) Review of WTP in developing countries using travel cost methods</p></li>
<li><p>(Ryan) Review of WTP methods for environmental quality</p></li>
<li><p>(Ryan) Method to estimate welfare effects (some exogenous shock)</p></li>
<li><p>(Raahil + Jovin + Matt) Determining the sample population criteria – important early step</p></li>
<li><p>(Raahil) Gathering data for population demographics (SHRUG, DHS, etc.)</p></li>
<li><p>Ways to annoate bibliography:</p>
<ul>
<li>paper title + journal, research question, research design (e.g. stated preference), data/sample size, result (i.e. WTP estimate)</li>
</ul></li>
<li><p>We need to think about how we contribute.</p></li>
<li><p>We need to think about sample selection.</p>
<ul>
<li>Criteria based on usage, overlap with protected areas, etc.</li>
<li>How do we remove tourists?</li>
<li>Who is the average person? Estimates will be sensitive</li>
</ul></li>
<li><p>Need to gridded map of ebird observations</p></li>
</ul>
</section>
<section id="next-scripts" class="level1">
<h1>Next scripts</h1>
<ol start="4" type="1">
<li>Merge attributes of the counterfactual sites. Need to collect site level data and/or individual level data
<ul>
<li>Site attributes: biodiversity, accessibility, ruggedness, congestion, temperature, rainfall, regional developmemnt</li>
<li>For first run, just do biodiversity.</li>
<li>Biodiversity comes from either diversity of birds or other attributes of ecosystem diversity</li>
<li>Expected biodiversity is why you go. So we can use historical measures of biodiversity</li>
</ul></li>
<li>Run the mixed logit to estimate the WTP values</li>
<li>Welfare analysis of WTP values</li>
</ol>
</section>
</div>
</div>
</div>
<div class="callout callout-style-default callout-note callout-titled">
<div class="callout-header d-flex align-content-center" data-bs-toggle="collapse" data-bs-target=".callout-13-contents" aria-controls="callout-13" aria-expanded="false" aria-label="Toggle callout">
<div class="callout-icon-container">
<i class='callout-icon'></i>
</div>
<div class="callout-title-container flex-fill">
02/28/25
</div>
<div class='callout-btn-toggle d-inline-block border-0 py-1 ps-1 pe-0 float-end'><i class='callout-toggle'></i></div>
</div>
<div id="callout-13" class="callout-13-contents callout-collapse collapse">
<div class="callout-body-container callout-body">
<blockquote>
<p>Raahil, Matt, Ryan</p>
</blockquote>
<section id="action-items-1" class="level2">
<h2>Action Items:</h2>
<ul>
<li>Ryan: Get the correct e-bird file from Matt (DONE)</li>
<li>Ryan: Merge code files into scripts folder. (DONE)</li>
<li>Ryan: Initate a document for literature review with template example (DONE)</li>
<li>We need to specalize and define tasks. Volunteer over email. Tasks: Lit review, sample selection, characterize the sample, theory person (DONE)</li>
<li>And set up a recurring meeting time. Need a poll. Try for next two weeks from now. (DONE)</li>
<li>Jovin: Provide your github username so Ryan can make you a collaborator on the github repo. (DONE)</li>
</ul>
</section>
<section id="funding" class="level2">
<h2>Funding:</h2>
<ul>
<li>No longer pursue IonE funding</li>
</ul>
</section>
<section id="coding" class="level2">
<h2>Coding:</h2>
<ul>
<li>Matt was able to replicate everything
<ul>
<li>Created a YAML to process file names</li>
<li>No longer needs new file paths</li>
</ul></li>
<li>Ryan stuck on loading data</li>
<li>Need to all use the same exact file download for e-bird</li>
<li>script 1: processes e-bird data, determines user homes, and calculates home for those who don’t report it.</li>
<li>script 2: Creates the hotspot observations. Measures distance from home to hotspot.</li>
<li>script 3: Most important. 12,000 hotspots so too many counterfactuals. This script reduces the counterfactual set by clustering hotspots nearby (likley parks) and only consider hotspots within a distance of home.</li>
<li>The final data set is a panel of person (user) ID by trip ID appended to all counterfactual trips for this user.</li>
<li>When we actually estimate this, we will need to plan for a no-trip counterfactual.</li>
<li>One of the main things we need to do is make the computation easier for us. Otherwise it becomes too computationally intensive.</li>
<li>Keep all the scripts in scripts folder.</li>
</ul>
</section>
<section id="data" class="level2">
<h2>Data:</h2>
<ul>
<li>We will need to bring in site attribute data</li>
<li>So data on landscape measurements.</li>
<li>Matt says that not all intermediate files are being save</li>
<li>Specifically, the master file is not saving</li>
<li>Need to identify our sample population
<ul>
<li>Need a careful (defensiable) data-driven criteria for sample</li>
<li>Needs to be regular, local users (not temporary or tourists)</li>
<li>Determine this based on usage frequency and mobility patterns</li>
<li>Consider overlaying protected areas polygons on top of the observations/hotspots</li>
<li>Buffers may create a censoring problem. So we would just test this against larger and larger buffers</li>
</ul></li>
<li>Need to find ways to characteristize the users (demographics) without having any information about them.
<ul>
<li>Important for welfare calculations.</li>
<li>Try to match with DHS data</li>
<li>Or overlap home location with SHRUG data set</li>
<li>Econometrically, take attribtues about where they live against the average for the country, how do high and low WTP birders fall within the distribution</li>
<li>Potentially use this as weights for welfare calculations.</li>
</ul></li>
</ul>
</section>
<section id="methods" class="level2">
<h2>Methods:</h2>
<ul>
<li>If we get a random shock to bird population, then we can think about the causal effect of increasing biodiversity</li>
</ul>
</section>
<section id="paperwriting" class="level2">
<h2>Paper/Writing:</h2>
<ul>
<li>For the ML cluster, will want to read up later to find out if there is a better way or more common practice in economics.</li>
<li>Create a lit review. Three groups:
<ol type="1">
<li>estimates of WTP in developing countries generally using the discrete choice travel cost method (Ted Miguel water quality paper).</li>
<li>WTP for environmental quality papers and what design they are using to estimate (e.g., contingent valuation, stated preferences – something from ecology)</li>
<li>Review of any updated versions of the mixed logit model – new adapations of the model from class</li>
</ol></li>
<li>For this: question, data used, design, and finding (dollar estimates would be great).</li>
<li>Create a document in the docs folder</li>
</ul>
</section>
</div>
</div>
</div>
<!-- -->
<div class="quarto-embedded-source-code">
<div class="sourceCode" id="cb2" data-shortcodes="false"><pre class="sourceCode markdown"><code class="sourceCode markdown"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="an">title:</span><span class="co"> &quot;The Economic Value of Biodiversity in India&quot;</span></span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="an">subtitle:</span><span class="co"> &quot;Running Notes&quot;</span></span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="an">author:</span><span class="co"> </span></span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a><span class="co">    - Raahil Madhok</span></span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a><span class="co">    - Matt Braaksma</span></span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a><span class="co">    - Ryan McWay</span></span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a><span class="co">    - Jovin Lasway</span></span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a><span class="an">toc:</span><span class="co"> true</span></span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a><span class="an">format:</span><span class="co"> </span></span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a><span class="co">    html:</span></span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a><span class="co">        theme: sandstone</span></span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a><span class="co">        html-math-method: katex</span></span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a><span class="co">        embed-resources: true</span></span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a><span class="co">        code-fold: true</span></span>
<span id="cb2-16"><a href="#cb2-16" aria-hidden="true" tabindex="-1"></a><span class="co">        code-tools: true</span></span>
<span id="cb2-17"><a href="#cb2-17" aria-hidden="true" tabindex="-1"></a><span class="co">        code-overflow: wrap</span></span>
<span id="cb2-18"><a href="#cb2-18" aria-hidden="true" tabindex="-1"></a><span class="co">    pdf:  </span></span>
<span id="cb2-19"><a href="#cb2-19" aria-hidden="true" tabindex="-1"></a><span class="co">        documentclass: article</span></span>
<span id="cb2-20"><a href="#cb2-20" aria-hidden="true" tabindex="-1"></a><span class="co">        papersize: letter</span></span>
<span id="cb2-21"><a href="#cb2-21" aria-hidden="true" tabindex="-1"></a><span class="an">number-sections:</span><span class="co"> false</span></span>
<span id="cb2-22"><a href="#cb2-22" aria-hidden="true" tabindex="-1"></a><span class="co"># jupyter: python3</span></span>
<span id="cb2-23"><a href="#cb2-23" aria-hidden="true" tabindex="-1"></a><span class="co"># NOTE: Quarto preview will only render the html file</span></span>
<span id="cb2-24"><a href="#cb2-24" aria-hidden="true" tabindex="-1"></a><span class="co">#       To render both html and pdf file run in terminal: </span></span>
<span id="cb2-25"><a href="#cb2-25" aria-hidden="true" tabindex="-1"></a><span class="co">#       quarto render docs/notes.qmd</span></span>
<span id="cb2-26"><a href="#cb2-26" aria-hidden="true" tabindex="-1"></a><span class="co">---</span></span>
<span id="cb2-27"><a href="#cb2-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-28"><a href="#cb2-28" aria-hidden="true" tabindex="-1"></a><span class="fu">## Project Description</span></span>
<span id="cb2-29"><a href="#cb2-29" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-30"><a href="#cb2-30" aria-hidden="true" tabindex="-1"></a>This project aims to produce the first measure of willingness to pay (WTP) for biodiversity. </span>
<span id="cb2-31"><a href="#cb2-31" aria-hidden="true" tabindex="-1"></a>This provides a monetary measure to value biodiversity which can be applied to a wide variety of applications in environmental economics and natural captial accounting. </span>
<span id="cb2-32"><a href="#cb2-32" aria-hidden="true" tabindex="-1"></a>The data relies on E-bird observations of diverse bird species observations. </span>
<span id="cb2-33"><a href="#cb2-33" aria-hidden="true" tabindex="-1"></a>The sample is limited to local, long-time users who are residents (not tourists) in India.</span>
<span id="cb2-34"><a href="#cb2-34" aria-hidden="true" tabindex="-1"></a>The methodology to calculate WTP relies on a revealed preference throughh the random utility model (RUM). </span>
<span id="cb2-35"><a href="#cb2-35" aria-hidden="true" tabindex="-1"></a>The RUM determines the value of seeking out diverse species of birds by comparing the value of alternative counterfactual bird siting locations and the cost to go to these counterfactuals. </span>
<span id="cb2-36"><a href="#cb2-36" aria-hidden="true" tabindex="-1"></a>Monetary value is determined through the RUM by estimating the travel cost of the reveal preference compared to counterfactual locations. </span>
<span id="cb2-37"><a href="#cb2-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-38"><a href="#cb2-38" aria-hidden="true" tabindex="-1"></a><span class="fu">## Notes</span></span>
<span id="cb2-39"><a href="#cb2-39" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-40"><a href="#cb2-40" aria-hidden="true" tabindex="-1"></a>For notes, we present the most recent first so that notes are chronologically most recent. </span>
<span id="cb2-41"><a href="#cb2-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-42"><a href="#cb2-42" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-43"><a href="#cb2-43" aria-hidden="true" tabindex="-1"></a>::: {.callout-note collapse=&quot;true&quot;}</span>
<span id="cb2-44"><a href="#cb2-44" aria-hidden="true" tabindex="-1"></a><span class="fu"># 9/12/2025</span></span>
<span id="cb2-45"><a href="#cb2-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-46"><a href="#cb2-46" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Raahil, Matt, Ryan, Jovin</span></span>
<span id="cb2-47"><a href="#cb2-47" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-48"><a href="#cb2-48" aria-hidden="true" tabindex="-1"></a><span class="fu"># TODO: </span></span>
<span id="cb2-49"><a href="#cb2-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-50"><a href="#cb2-50" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt**: Diagnositics for missing species richness</span>
<span id="cb2-51"><a href="#cb2-51" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt/Jovin**: Migration component for Shift-share</span>
<span id="cb2-52"><a href="#cb2-52" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Ryan/Jovin**: Computing Shift-Share</span>
<span id="cb2-53"><a href="#cb2-53" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**All**: Review new working paper (has new IV)</span>
<span id="cb2-54"><a href="#cb2-54" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-55"><a href="#cb2-55" aria-hidden="true" tabindex="-1"></a><span class="fu">## Agenda</span></span>
<span id="cb2-56"><a href="#cb2-56" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-57"><a href="#cb2-57" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Missing information for species richness by season</span>
<span id="cb2-58"><a href="#cb2-58" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Core issue coming from the imputation with BirdLife data leading to negative coefficient</span>
<span id="cb2-59"><a href="#cb2-59" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Several hotspot clusters are missing seasonal information</span>
<span id="cb2-60"><a href="#cb2-60" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Means that noone is visiting these hotspots previously (e.g., rarely visited)</span>
<span id="cb2-61"><a href="#cb2-61" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Diagnostic: Number of trips per hotspot (confirm they are hotspots)</span>
<span id="cb2-62"><a href="#cb2-62" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Number of distinct time periods with observations for each hotspot</span>
<span id="cb2-63"><a href="#cb2-63" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>Migration for Shift-share</span>
<span id="cb2-64"><a href="#cb2-64" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Classify birds are migratory or not</span>
<span id="cb2-65"><a href="#cb2-65" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Remove species with missing migratory classification</span>
<span id="cb2-66"><a href="#cb2-66" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Keep migratory species to determine migration connections to regions</span>
<span id="cb2-67"><a href="#cb2-67" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Jovin has successfully matched a lot of these (fuzzy matching)</span>
<span id="cb2-68"><a href="#cb2-68" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-69"><a href="#cb2-69" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-70"><a href="#cb2-70" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-71"><a href="#cb2-71" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-72"><a href="#cb2-72" aria-hidden="true" tabindex="-1"></a>::: {.callout-note collapse=&quot;true&quot;}</span>
<span id="cb2-73"><a href="#cb2-73" aria-hidden="true" tabindex="-1"></a><span class="fu"># 8/27/2025</span></span>
<span id="cb2-74"><a href="#cb2-74" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-75"><a href="#cb2-75" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Ryan, Jovin</span></span>
<span id="cb2-76"><a href="#cb2-76" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-77"><a href="#cb2-77" aria-hidden="true" tabindex="-1"></a><span class="fu"># TODO: </span></span>
<span id="cb2-78"><a href="#cb2-78" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt**: We need to meet with you to discuss working on this with MSI</span>
<span id="cb2-79"><a href="#cb2-79" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Jovin**: Responsible for Shift</span>
<span id="cb2-80"><a href="#cb2-80" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Ryan**: Responsible for Share</span>
<span id="cb2-81"><a href="#cb2-81" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-82"><a href="#cb2-82" aria-hidden="true" tabindex="-1"></a><span class="fu">## Plan to Estimate Shift-Share</span></span>
<span id="cb2-83"><a href="#cb2-83" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-84"><a href="#cb2-84" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Merge the Species range data with the migratory species data</span>
<span id="cb2-85"><a href="#cb2-85" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Species range data: base-data/species/BOTW_2024_2.gpkg</span>
<span id="cb2-86"><a href="#cb2-86" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Migratory species data: base-data/species/species_list_categorized.csv</span>
<span id="cb2-87"><a href="#cb2-87" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Merge by: scientific_name</span>
<span id="cb2-88"><a href="#cb2-88" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Result: species level data with range map and classifier for migratory</span>
<span id="cb2-89"><a href="#cb2-89" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>For each country j, calculate $\gamma_j$</span>
<span id="cb2-90"><a href="#cb2-90" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-91"><a href="#cb2-91" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-92"><a href="#cb2-92" aria-hidden="true" tabindex="-1"></a>\gamma_j = \sum_{s =\text{total num of species in j}}^S \frac{\text{species range overlap India}_{sj}}{S_j}</span>
<span id="cb2-93"><a href="#cb2-93" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-94"><a href="#cb2-94" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-95"><a href="#cb2-95" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Count the number of species $s$ in $j$</span>
<span id="cb2-96"><a href="#cb2-96" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Species range data: base-data/species/BOTW_2024_2.gpkg</span>
<span id="cb2-97"><a href="#cb2-97" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>India Borders: data/shp/district-2011</span>
<span id="cb2-98"><a href="#cb2-98" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Need to make one shape (spatial union) so any intersection with India shape</span>
<span id="cb2-99"><a href="#cb2-99" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Indicator for each species if any overlap of species with India borders</span>
<span id="cb2-100"><a href="#cb2-100" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-101"><a href="#cb2-101" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Spatial overlap of species $s$ range over the country border of India. This will create some fraction. </span>
<span id="cb2-102"><a href="#cb2-102" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Calculate sqkm for India shapefile one time (part 1)</span>
<span id="cb2-103"><a href="#cb2-103" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Spatial inner join of range $s$ on India. </span>
<span id="cb2-104"><a href="#cb2-104" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Calculate sqkm for the inner join (part 2)</span>
<span id="cb2-105"><a href="#cb2-105" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Divide part 2 by part 1 to get the fraction of overlap for species range over India borders</span>
<span id="cb2-106"><a href="#cb2-106" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-107"><a href="#cb2-107" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Combine to make $\gamma_j$ using the weighted sum $\sum_s$</span>
<span id="cb2-108"><a href="#cb2-108" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Spatial intersection of species range $s$ with countries in the flu dataset</span>
<span id="cb2-109"><a href="#cb2-109" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Create the weight $S_j$ by counting the number of indicators for intersecting ranges with $j$</span>
<span id="cb2-110"><a href="#cb2-110" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Create $\gamma_j$ for each $j$ by summing fractions in (2) divided by weight $S_j$</span>
<span id="cb2-111"><a href="#cb2-111" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Country $j$ Shapefiles: TODO.</span>
<span id="cb2-112"><a href="#cb2-112" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-113"><a href="#cb2-113" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Create the shift by merging $\gamma_j$ with the flu data</span>
<span id="cb2-114"><a href="#cb2-114" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Flu Data: data/clean/flu-data.csv</span>
<span id="cb2-115"><a href="#cb2-115" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Result is time series by t</span>
<span id="cb2-116"><a href="#cb2-116" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-117"><a href="#cb2-117" aria-hidden="true" tabindex="-1"></a>Shift_t = \sum_{j = country}^J flu_{jt} \times \gamma_j</span>
<span id="cb2-118"><a href="#cb2-118" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-119"><a href="#cb2-119" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-120"><a href="#cb2-120" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Create the share for each hotspot t</span>
<span id="cb2-121"><a href="#cb2-121" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-122"><a href="#cb2-122" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-123"><a href="#cb2-123" aria-hidden="true" tabindex="-1"></a>Share_d =  \sum_s \frac{\text{Indicator for migratory}_{sd}}{S_d}</span>
<span id="cb2-124"><a href="#cb2-124" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-125"><a href="#cb2-125" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-126"><a href="#cb2-126" aria-hidden="true" tabindex="-1"></a><span class="in">    1. Start with e-bird data. For each hotspot $d$ get the number of unique species observed in the data</span></span>
<span id="cb2-127"><a href="#cb2-127" aria-hidden="true" tabindex="-1"></a><span class="in">        - e-bird data: Ask Matt what is the best way to do this given the current workflow. </span></span>
<span id="cb2-128"><a href="#cb2-128" aria-hidden="true" tabindex="-1"></a><span class="in">    2. Merge in the migratory data on these species for each hotspot $d$</span></span>
<span id="cb2-129"><a href="#cb2-129" aria-hidden="true" tabindex="-1"></a><span class="in">        - Migratory species data: base-data/species/species_list_categorized.csv</span></span>
<span id="cb2-130"><a href="#cb2-130" aria-hidden="true" tabindex="-1"></a><span class="in">        - Merge by: scientific_name</span></span>
<span id="cb2-131"><a href="#cb2-131" aria-hidden="true" tabindex="-1"></a><span class="in">    3. Create the share variable as weighted sum for hotspot $d$</span></span>
<span id="cb2-132"><a href="#cb2-132" aria-hidden="true" tabindex="-1"></a><span class="in">        - For hotspot $d$, count indicator for migratory. </span></span>
<span id="cb2-133"><a href="#cb2-133" aria-hidden="true" tabindex="-1"></a><span class="in">        - For hotspot $d$, get count of unique species $S_d$</span></span>
<span id="cb2-134"><a href="#cb2-134" aria-hidden="true" tabindex="-1"></a><span class="in">        - Then construct sum of the fractions. </span></span>
<span id="cb2-135"><a href="#cb2-135" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-136"><a href="#cb2-136" aria-hidden="true" tabindex="-1"></a><span class="ss">5. </span>Create the IV by multiplying the shift by the share for each hotspot d</span>
<span id="cb2-137"><a href="#cb2-137" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-138"><a href="#cb2-138" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-139"><a href="#cb2-139" aria-hidden="true" tabindex="-1"></a>IV_{dt} = shift_t \times share_d</span>
<span id="cb2-140"><a href="#cb2-140" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-141"><a href="#cb2-141" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-142"><a href="#cb2-142" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-143"><a href="#cb2-143" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-144"><a href="#cb2-144" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-145"><a href="#cb2-145" aria-hidden="true" tabindex="-1"></a>::: {.callout-note collapse=&quot;true&quot;}</span>
<span id="cb2-146"><a href="#cb2-146" aria-hidden="true" tabindex="-1"></a><span class="fu"># 8/15/2025</span></span>
<span id="cb2-147"><a href="#cb2-147" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-148"><a href="#cb2-148" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Ryan, Matt, Raahil, Jovin</span></span>
<span id="cb2-149"><a href="#cb2-149" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-150"><a href="#cb2-150" aria-hidden="true" tabindex="-1"></a><span class="fu"># TODO: </span></span>
<span id="cb2-151"><a href="#cb2-151" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt**: Demean data for State x month fixed effect and user fixed effects </span>
<span id="cb2-152"><a href="#cb2-152" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt**: Resolve imputing species richness by estimating correlation of e-bird to bird life data and deflate value</span>
<span id="cb2-153"><a href="#cb2-153" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>**Matt**: Degree of Imputation: Summary stats on % of missing in observed and counterfactual data</span>
<span id="cb2-154"><a href="#cb2-154" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Jovin**: Constructing the shift-share IV</span>
<span id="cb2-155"><a href="#cb2-155" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-156"><a href="#cb2-156" aria-hidden="true" tabindex="-1"></a><span class="fu">## Agenda</span></span>
<span id="cb2-157"><a href="#cb2-157" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-158"><a href="#cb2-158" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Estimation of WTP</span>
<span id="cb2-159"><a href="#cb2-159" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>negative values came from mismatched polygons</span>
<span id="cb2-160"><a href="#cb2-160" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Difference from the clipped calipers instead of the original simple circles</span>
<span id="cb2-161"><a href="#cb2-161" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>weeks sometimes were only 53 (not 54). Solved with iso-week formating that allows for more merging of site attributes</span>
<span id="cb2-162"><a href="#cb2-162" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>Data issue</span>
<span id="cb2-163"><a href="#cb2-163" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>we don&#39;t observe species richness</span>
<span id="cb2-164"><a href="#cb2-164" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>So we are imputing with range maps</span>
<span id="cb2-165"><a href="#cb2-165" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>When there is missing, we currently fill with species map</span>
<span id="cb2-166"><a href="#cb2-166" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Possible species ranges is always higher than recorded value from e-birders</span>
<span id="cb2-167"><a href="#cb2-167" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>So the imputed values always are super high when missing, and then this makes our estimates really weird.</span>
<span id="cb2-168"><a href="#cb2-168" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>This penalizes the choice. Because counterfactual choice has huge species richness</span>
<span id="cb2-169"><a href="#cb2-169" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Potential Solution: replace missing with historical average (or minimum average) for each district or state in the same time period. </span>
<span id="cb2-170"><a href="#cb2-170" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Impute that for hotspots with missing data </span>
<span id="cb2-171"><a href="#cb2-171" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Potential Solution: correlation of bird life data and e-bird observations</span>
<span id="cb2-172"><a href="#cb2-172" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Then we can just deflate imputed value that rescales this to the e-bird values</span>
<span id="cb2-173"><a href="#cb2-173" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>What is the main source of missing value? </span>
<span id="cb2-174"><a href="#cb2-174" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>want summary stats on % of missing in observed and counterfactual data</span>
<span id="cb2-175"><a href="#cb2-175" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Could make this seasonal species richness for imputed value</span>
<span id="cb2-176"><a href="#cb2-176" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>Fixed Effects Issue</span>
<span id="cb2-177"><a href="#cb2-177" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>NA for standard errors when adding individual fixed effects</span>
<span id="cb2-178"><a href="#cb2-178" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Running with python on MSI</span>
<span id="cb2-179"><a href="#cb2-179" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Could attempt to manually demean data (user fixed effects) before running</span>
<span id="cb2-180"><a href="#cb2-180" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>State x month fixed effect and user fixed effects </span>
<span id="cb2-181"><a href="#cb2-181" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Robustness by district x month fixed effects </span>
<span id="cb2-182"><a href="#cb2-182" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>This will just be more intense</span>
<span id="cb2-183"><a href="#cb2-183" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>mlogit does not allow for fixed effects</span>
<span id="cb2-184"><a href="#cb2-184" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>The key is to demean within each group</span>
<span id="cb2-185"><a href="#cb2-185" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>hdfe is package in Stata</span>
<span id="cb2-186"><a href="#cb2-186" aria-hidden="true" tabindex="-1"></a><span class="ss">4. </span>Instrumental Variables</span>
<span id="cb2-187"><a href="#cb2-187" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Have a map data for bird migration routes</span>
<span id="cb2-188"><a href="#cb2-188" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Only found 2 species public for India </span>
<span id="cb2-189"><a href="#cb2-189" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>There are other species we can request access to</span>
<span id="cb2-190"><a href="#cb2-190" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Data avaliable for avian flu outbreak</span>
<span id="cb2-191"><a href="#cb2-191" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>H5N1</span>
<span id="cb2-192"><a href="#cb2-192" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>2005 - 2025</span>
<span id="cb2-193"><a href="#cb2-193" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Combining the two data sets we can assign outbreaks to specific species that migrate to India</span>
<span id="cb2-194"><a href="#cb2-194" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Human cases of avian flu deaths should be controlled for in IV estimation</span>
<span id="cb2-195"><a href="#cb2-195" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>If we have a shift-share IV, do we need to know the flyways?</span>
<span id="cb2-196"><a href="#cb2-196" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>If this is the only channel, then we don&#39;t need to actually need to know the origin of the bird </span>
<span id="cb2-197"><a href="#cb2-197" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-198"><a href="#cb2-198" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-199"><a href="#cb2-199" aria-hidden="true" tabindex="-1"></a>IV_{dt} = shift_t \times share_d</span>
<span id="cb2-200"><a href="#cb2-200" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-201"><a href="#cb2-201" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-202"><a href="#cb2-202" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-203"><a href="#cb2-203" aria-hidden="true" tabindex="-1"></a>IV_{dt} = \text{Wild Avian Flu}_t \times \text{% of species pool from migration}_d</span>
<span id="cb2-204"><a href="#cb2-204" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-205"><a href="#cb2-205" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-206"><a href="#cb2-206" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-207"><a href="#cb2-207" aria-hidden="true" tabindex="-1"></a>Shift_t = \sum_{j = country}^J flu_{jt} \times \gamma_j</span>
<span id="cb2-208"><a href="#cb2-208" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-209"><a href="#cb2-209" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-210"><a href="#cb2-210" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-211"><a href="#cb2-211" aria-hidden="true" tabindex="-1"></a>\gamma_j = \sum_{s =\text{total num of species in j}}^S \frac{\text{species range overlap India}_{sj}}{S_j}</span>
<span id="cb2-212"><a href="#cb2-212" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-213"><a href="#cb2-213" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-214"><a href="#cb2-214" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Assumes a uniform distribution of species across their species range</span>
<span id="cb2-215"><a href="#cb2-215" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>This is an improvement over distance from India as the weight</span>
<span id="cb2-216"><a href="#cb2-216" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-217"><a href="#cb2-217" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-218"><a href="#cb2-218" aria-hidden="true" tabindex="-1"></a>Share_d =  \sum_s \frac{\text{Indicator for migratory}_{sd}}{S_d}</span>
<span id="cb2-219"><a href="#cb2-219" aria-hidden="true" tabindex="-1"></a>$$</span>
<span id="cb2-220"><a href="#cb2-220" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-221"><a href="#cb2-221" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-222"><a href="#cb2-222" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-223"><a href="#cb2-223" aria-hidden="true" tabindex="-1"></a>::: {.callout-note collapse=&quot;true&quot;}</span>
<span id="cb2-224"><a href="#cb2-224" aria-hidden="true" tabindex="-1"></a><span class="fu"># 8/1/2025</span></span>
<span id="cb2-225"><a href="#cb2-225" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-226"><a href="#cb2-226" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Ryan, Matt, Raahil</span></span>
<span id="cb2-227"><a href="#cb2-227" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-228"><a href="#cb2-228" aria-hidden="true" tabindex="-1"></a><span class="fu"># TODO: </span></span>
<span id="cb2-229"><a href="#cb2-229" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt**: Re-estimating WTP using new sample of users (expanded)</span>
<span id="cb2-230"><a href="#cb2-230" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt**: Adding a Visual: Attempting to run the simple regression of biodiversity and hotspot visitations</span>
<span id="cb2-231"><a href="#cb2-231" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Jovin**: Complete the IV design by finding accompanying data. Maps of migration patterns. Papers to cite for this phenomenon.</span>
<span id="cb2-232"><a href="#cb2-232" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Raahil/Ryan**: Write up the model in LaTeX</span>
<span id="cb2-233"><a href="#cb2-233" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-234"><a href="#cb2-234" aria-hidden="true" tabindex="-1"></a><span class="fu">## Agenda</span></span>
<span id="cb2-235"><a href="#cb2-235" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-236"><a href="#cb2-236" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Figures describing variation </span>
<span id="cb2-237"><a href="#cb2-237" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Ryan presented some post estimation figures we tabled for later</span>
<span id="cb2-238"><a href="#cb2-238" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Matt presented some maps showing variation over space</span>
<span id="cb2-239"><a href="#cb2-239" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Raahil suggested we produce a margins plots. We run an endogenous TWFE model of species richness on number of visitors. Then we plot the marginal effects at different levels of biodiversity. </span>
<span id="cb2-240"><a href="#cb2-240" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>First estimates of WTP using super users </span>
<span id="cb2-241"><a href="#cb2-241" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Matt is able to run the conditional logit model. Not able to run the mixed logit model</span>
<span id="cb2-242"><a href="#cb2-242" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Our $\alpha$ for price has the wrong sign. This makes biodiversity WTP a negative value</span>
<span id="cb2-243"><a href="#cb2-243" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>We think that the sample may be too restrictive resulting in weird results </span>
<span id="cb2-244"><a href="#cb2-244" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Additionally, Matt is working through some data issues in the counterfactual data set that is dropping 1.1% of site attributes</span>
<span id="cb2-245"><a href="#cb2-245" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>And there is colinearity amongst site attributes when running the conditional logit, so they are currently dropped in the model run. </span>
<span id="cb2-246"><a href="#cb2-246" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>Modeling updates using IV method</span>
<span id="cb2-247"><a href="#cb2-247" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Ryan has a clear idea of how to model the IV into our mixed logit model </span>
<span id="cb2-248"><a href="#cb2-248" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>The key is to rely on Jacob Bradt&#39;s model using control functions. </span>
<span id="cb2-249"><a href="#cb2-249" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>We estimate the first stage linear regression. Residualize. And use the residuals in our discrete choice method (mixed logit). </span>
<span id="cb2-250"><a href="#cb2-250" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>We will need to bootstrap the 2nd stage in our to recover standard errors. </span>
<span id="cb2-251"><a href="#cb2-251" aria-hidden="true" tabindex="-1"></a><span class="ss">4. </span>Bird Migration Instruments</span>
<span id="cb2-252"><a href="#cb2-252" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>(i) Migration patterns: From the literature I&#39;ve compiled so far:</span>
<span id="cb2-253"><a href="#cb2-253" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Peak arrivals: August - November</span>
<span id="cb2-254"><a href="#cb2-254" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Peak departures: February - April</span>
<span id="cb2-255"><a href="#cb2-255" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Low migration periods: May - July, and December - January</span>
<span id="cb2-256"><a href="#cb2-256" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>The Inbound migration: Birds arrive in India primarily from the northern breeding grounds, especially in Mongolia, Kazakhstan, and Siberia, during the August-November window to overwinter in wetlands and coastal areas.</span>
<span id="cb2-257"><a href="#cb2-257" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>The Outbound migration: Birds depart February-April, returning to their breeding sites for nesting.</span>
<span id="cb2-258"><a href="#cb2-258" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>(ii) Drivers of bird migration and IV framework: Ecology literature indicates that migration along the Central Asian Flyway is mainly driven by (i) Ecological reasons, india&#39;s wetlands, lakes, and agricultural landscapes provide abundant food and shelter during winter, (ii) Climatic reasons: Wrmer winter conditions compared to the northern latitudes reduce suruval stress (iii) Stopover importance: Key sites such as Chilika, Keoladeo, Pong Dam, etc serve as staging areas for fueling before onward flights.</span>
<span id="cb2-259"><a href="#cb2-259" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Recent literature uses environmental shocks as instruments (Noack et al., 2024; Meng et al, 2025). For India&#39;s bird migration context, two IV strategies appear promising.</span>
<span id="cb2-260"><a href="#cb2-260" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Weather shock IV: Baseline district exposure (from our eBird) interacting with NVDI and weather anomalies in the breeding grounds.</span>
<span id="cb2-261"><a href="#cb2-261" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Disease shock IV: I find this is more interesting because a recent work by Yang et al., 2024 at Nature Communications finds avian influenza outbreaks do not trigger migration but do affect survival and presence of birds at the Indian stopovers to a very large extent. This makes it a credible instrument because: (i) outbreaks are exogenous to the local economic variables and (ii) outbreak timing is tied to a peak of migration window/CAF (Aug-Nov, Feb-Apr) in India, allowing shocks to coincide with relevant bird presence.</span>
<span id="cb2-262"><a href="#cb2-262" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>(iii) Data collection: Potential outbreak data sources, shapefiles include (i) FAO EMPRES-i+; (ii) WOAH/WAHIS, which tracks the outbreak locations, date, disease type, host species, and status. Lastly, GISAID shapefile data, which is an avian influenza sequence database, tracks the outbreak locations (used in Elbe &amp; Buckland-Merrett, 2017, Global Challenges Journal; Yang et al, 2024, Nature Communications).</span>
<span id="cb2-263"><a href="#cb2-263" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>(iv) Next step: Performing a spatial join between outbreak data and our ebird-based hotspots, aggregating outbreak counts to the district/hotspot-year panel level. I think this will allow us to construct the avian influenza IV as = Baseline exposure in district/hotspot d  x outbreak shock at time t.</span>
<span id="cb2-264"><a href="#cb2-264" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-265"><a href="#cb2-265" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-266"><a href="#cb2-266" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-267"><a href="#cb2-267" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-268"><a href="#cb2-268" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-269"><a href="#cb2-269" aria-hidden="true" tabindex="-1"></a>::: {.callout-note collapse=&quot;true&quot;}</span>
<span id="cb2-270"><a href="#cb2-270" aria-hidden="true" tabindex="-1"></a><span class="fu"># 6/2/2025</span></span>
<span id="cb2-271"><a href="#cb2-271" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-272"><a href="#cb2-272" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Ryan, Matt, Jovin, Raahil</span></span>
<span id="cb2-273"><a href="#cb2-273" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-274"><a href="#cb2-274" aria-hidden="true" tabindex="-1"></a><span class="fu"># TODO: </span></span>
<span id="cb2-275"><a href="#cb2-275" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt**: Finish species richness site attribute (cluster-time level)</span>
<span id="cb2-276"><a href="#cb2-276" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt**: Finish computing congestion site attribute</span>
<span id="cb2-277"><a href="#cb2-277" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt**: Check why sample size so big. Check number of elements in the choice set for validation.</span>
<span id="cb2-278"><a href="#cb2-278" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Matt/Ryan**: Compute species richness in radius *around home* (varying at same time unit as other species richness attribute)</span>
<span id="cb2-279"><a href="#cb2-279" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Jovin**: Fix travel cost. I suggest checking it over with Matt/Ryan</span>
<span id="cb2-280"><a href="#cb2-280" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>**Raahil/Ryan**: Work through paper framing</span>
<span id="cb2-281"><a href="#cb2-281" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-282"><a href="#cb2-282" aria-hidden="true" tabindex="-1"></a><span class="fu">## Agenda</span></span>
<span id="cb2-283"><a href="#cb2-283" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-284"><a href="#cb2-284" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Species Richness </span>
<span id="cb2-285"><a href="#cb2-285" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Maps for imputation can&#39;t have the shannon index. Because we don&#39;t have abundance counts</span>
<span id="cb2-286"><a href="#cb2-286" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>But we can do an index for the e-bird data hotspots</span>
<span id="cb2-287"><a href="#cb2-287" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Estimates of bird life and expected species richness </span>
<span id="cb2-288"><a href="#cb2-288" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Monthly values or seasonal values </span>
<span id="cb2-289"><a href="#cb2-289" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-290"><a href="#cb2-290" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-291"><a href="#cb2-291" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>Other Site Attributes </span>
<span id="cb2-292"><a href="#cb2-292" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Have one site data set for temp, rain, and tree cover</span>
<span id="cb2-293"><a href="#cb2-293" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>This is at the hotspot cluster level </span>
<span id="cb2-294"><a href="#cb2-294" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-295"><a href="#cb2-295" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>Congestion </span>
<span id="cb2-296"><a href="#cb2-296" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Ready to go code, just not run yet</span>
<span id="cb2-297"><a href="#cb2-297" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span><span class="sc">\#</span> of users at hotspot cluster monthly</span>
<span id="cb2-298"><a href="#cb2-298" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-299"><a href="#cb2-299" aria-hidden="true" tabindex="-1"></a><span class="ss">4. </span>Travel Cost</span>
<span id="cb2-300"><a href="#cb2-300" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Jovin has choice set and code that estmates this for a first attempt </span>
<span id="cb2-301"><a href="#cb2-301" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Matched to GDP data to impute wages </span>
<span id="cb2-302"><a href="#cb2-302" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Estimate Euclidean distance between households and hotspot clusters </span>
<span id="cb2-303"><a href="#cb2-303" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Final data for RUM should be choice set dimensions (for our checks)</span>
<span id="cb2-304"><a href="#cb2-304" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-305"><a href="#cb2-305" aria-hidden="true" tabindex="-1"></a><span class="ss">5. </span>Measure of Experience at the Individual Level </span>
<span id="cb2-306"><a href="#cb2-306" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Counts of trips (counter)</span>
<span id="cb2-307"><a href="#cb2-307" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Count for <span class="sc">\#</span> of different hotspots (counter)</span>
<span id="cb2-308"><a href="#cb2-308" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>So ratio tells us if you have  preference for location (&quot;Habit Formation&quot;). </span>
<span id="cb2-309"><a href="#cb2-309" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Low ratio is little location preference</span>
<span id="cb2-310"><a href="#cb2-310" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>High ratio is high location preference</span>
<span id="cb2-311"><a href="#cb2-311" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-312"><a href="#cb2-312" aria-hidden="true" tabindex="-1"></a><span class="ss">6. </span>Conceptual</span>
<span id="cb2-313"><a href="#cb2-313" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Birds vs. non-bird still valued as travel cost. So both are still measures of nature recreation. </span>
<span id="cb2-314"><a href="#cb2-314" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Monetizing value &gt;&gt; Add in contribute to GEP or ecosystem service value literature. </span>
<span id="cb2-315"><a href="#cb2-315" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Novelty: Why is WTP lower in Developing Countries? Welfare for environmental quality in envirodevonomics framework (Env-Dev framing)</span>
<span id="cb2-316"><a href="#cb2-316" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-317"><a href="#cb2-317" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-318"><a href="#cb2-318" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-319"><a href="#cb2-319" aria-hidden="true" tabindex="-1"></a>::: {.callout-note collapse=&quot;true&quot;}</span>
<span id="cb2-320"><a href="#cb2-320" aria-hidden="true" tabindex="-1"></a><span class="fu"># 5/16/2025</span></span>
<span id="cb2-321"><a href="#cb2-321" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-322"><a href="#cb2-322" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Ryan, Matt, Jovin, Raahil</span></span>
<span id="cb2-323"><a href="#cb2-323" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-324"><a href="#cb2-324" aria-hidden="true" tabindex="-1"></a><span class="fu"># TODO: </span></span>
<span id="cb2-325"><a href="#cb2-325" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Seperate meetings for Matt/Jovin (coding) and Ryan/Raahil (conception) for our specializations</span>
<span id="cb2-326"><a href="#cb2-326" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-327"><a href="#cb2-327" aria-hidden="true" tabindex="-1"></a><span class="fu">## Agenda</span></span>
<span id="cb2-328"><a href="#cb2-328" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-329"><a href="#cb2-329" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Finalize the site attribute data and create the master dataset</span>
<span id="cb2-330"><a href="#cb2-330" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Matt and Ryan -- complete by next meeting (May 16)</span>
<span id="cb2-331"><a href="#cb2-331" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We have monthly temperature and rainfall</span>
<span id="cb2-332"><a href="#cb2-332" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We have annual tree cover data</span>
<span id="cb2-333"><a href="#cb2-333" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>These are as zonal statistics (average for 10KM radius of hotspot)</span>
<span id="cb2-334"><a href="#cb2-334" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>In E-bird, we have the average distance walked around hotspot</span>
<span id="cb2-335"><a href="#cb2-335" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Or do we avoid this work because temp, rain, and tree cover does not vary much at such a small spatial scale.</span>
<span id="cb2-336"><a href="#cb2-336" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Temp, rainfall, and tree are at higher resolutions. So as long as the radius is less than the input data this will not vary. </span>
<span id="cb2-337"><a href="#cb2-337" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We need to think more about mapping of hotspots and hotspot clusters. </span>
<span id="cb2-338"><a href="#cb2-338" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Need to add a fourth site attribute: congestion</span>
<span id="cb2-339"><a href="#cb2-339" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>How frequently other users are using this site? </span>
<span id="cb2-340"><a href="#cb2-340" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Aggregate number of user trips for each hotspot but will need to do this as a lag (week before)</span>
<span id="cb2-341"><a href="#cb2-341" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>So this is expected congestion. </span>
<span id="cb2-342"><a href="#cb2-342" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We need to transform this into a panel of counterfactual sets appended to the observed trips. </span>
<span id="cb2-343"><a href="#cb2-343" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Not balanced across counterfactual trips and time </span>
<span id="cb2-344"><a href="#cb2-344" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-345"><a href="#cb2-345" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-346"><a href="#cb2-346" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>Calculate travel costs -- see notes for guidance; this will be added to the site attribute script</span>
<span id="cb2-347"><a href="#cb2-347" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>For GDP per capita data, use gridded data from here and extract the value closest to each users&#39; home. Then use the formula from Kolstoe/cameron or lloyd-smith to calculate travel costs. </span>
<span id="cb2-348"><a href="#cb2-348" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Jovin -- complete by next meeting (May 16)</span>
<span id="cb2-349"><a href="#cb2-349" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Collected 2014-2024 GDP per capita data</span>
<span id="cb2-350"><a href="#cb2-350" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Laptop trouble estimating, working with MPC.</span>
<span id="cb2-351"><a href="#cb2-351" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Could run the travel cost with a small random sample to make this managable for Jovin&#39;s laptop capacity.</span>
<span id="cb2-352"><a href="#cb2-352" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Can re-run the pipeline so we are running for only frequent users (1.6M instead of the 3M)</span>
<span id="cb2-353"><a href="#cb2-353" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-354"><a href="#cb2-354" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>Species richness </span>
<span id="cb2-355"><a href="#cb2-355" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Matt and Ryan -- bring three ideas for next meeting (May 16)</span>
<span id="cb2-356"><a href="#cb2-356" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Close to being done (Matt taking the lead)</span>
<span id="cb2-357"><a href="#cb2-357" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Taking raw e-bird data and getting monthly counts</span>
<span id="cb2-358"><a href="#cb2-358" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Need to translate from hotspots to hotspot clusters</span>
<span id="cb2-359"><a href="#cb2-359" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Species ranges (for missing) shapefiles are crashing the code.</span>
<span id="cb2-360"><a href="#cb2-360" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>All of this is to get a species at a hotspot so we can turn this into a richness index </span>
<span id="cb2-361"><a href="#cb2-361" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Will want this merged as richness for same month/week from previous year</span>
<span id="cb2-362"><a href="#cb2-362" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>In the app, you can see in real-time species count from their favorite places. </span>
<span id="cb2-363"><a href="#cb2-363" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>So likely quite responsive. Last week is best</span>
<span id="cb2-364"><a href="#cb2-364" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Can also do monthly to replicate results from month last year from Kolsto paper method. </span>
<span id="cb2-365"><a href="#cb2-365" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We aim to use the simpson and shanon index from <span class="in">`vegan`</span> package</span>
<span id="cb2-366"><a href="#cb2-366" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-367"><a href="#cb2-367" aria-hidden="true" tabindex="-1"></a><span class="ss">4. </span>Individual Attributes</span>
<span id="cb2-368"><a href="#cb2-368" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>With individual fixed effects, we partial out any time-invariant</span>
<span id="cb2-369"><a href="#cb2-369" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>From the e-bird data:</span>
<span id="cb2-370"><a href="#cb2-370" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>How frequently you went in the past year</span>
<span id="cb2-371"><a href="#cb2-371" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Average distance traveled in the past year </span>
<span id="cb2-372"><a href="#cb2-372" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Is there any measure of learning over time? Value of recreation is not static</span>
<span id="cb2-373"><a href="#cb2-373" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Experience index</span>
<span id="cb2-374"><a href="#cb2-374" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Cumulative experience (simple counter of trips up unti that point)</span>
<span id="cb2-375"><a href="#cb2-375" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Could weight this by how much you walk, and taxonomy of bird level data (raw text file of trip-species level)</span>
<span id="cb2-376"><a href="#cb2-376" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Could collapse species count to trip level. </span>
<span id="cb2-377"><a href="#cb2-377" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-378"><a href="#cb2-378" aria-hidden="true" tabindex="-1"></a><span class="fu">## Project Management</span></span>
<span id="cb2-379"><a href="#cb2-379" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-380"><a href="#cb2-380" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Meetings need to be in person. </span>
<span id="cb2-381"><a href="#cb2-381" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Jovin update meeting calendar </span>
<span id="cb2-382"><a href="#cb2-382" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Start assigning tasks instead of asking for volunteers</span>
<span id="cb2-383"><a href="#cb2-383" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Need to come up with a specialization of tasks. </span>
<span id="cb2-384"><a href="#cb2-384" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Coding and Data: Matt (lead), Jovin, Ryan (assist)</span>
<span id="cb2-385"><a href="#cb2-385" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Conceptualize: Ryan (lead), and Raahil</span>
<span id="cb2-386"><a href="#cb2-386" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>How our model departs from workhouse. </span>
<span id="cb2-387"><a href="#cb2-387" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Literature review for our novelty</span>
<span id="cb2-388"><a href="#cb2-388" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Some tasks: </span>
<span id="cb2-389"><a href="#cb2-389" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Google maps driving time</span>
<span id="cb2-390"><a href="#cb2-390" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>inaturalist species richness mapping</span>
<span id="cb2-391"><a href="#cb2-391" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-392"><a href="#cb2-392" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-393"><a href="#cb2-393" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-394"><a href="#cb2-394" aria-hidden="true" tabindex="-1"></a>::: {.callout-note collapse=&quot;true&quot;}</span>
<span id="cb2-395"><a href="#cb2-395" aria-hidden="true" tabindex="-1"></a><span class="fu"># 5/15/2025</span></span>
<span id="cb2-396"><a href="#cb2-396" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-397"><a href="#cb2-397" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Ryan, Matt</span></span>
<span id="cb2-398"><a href="#cb2-398" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-399"><a href="#cb2-399" aria-hidden="true" tabindex="-1"></a><span class="fu">## Coding Notes</span></span>
<span id="cb2-400"><a href="#cb2-400" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Ryan is taking lead on estimating travel cost. </span>
<span id="cb2-401"><a href="#cb2-401" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We need a seperate hotspots shapefile for running zonal stats </span>
<span id="cb2-402"><a href="#cb2-402" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We need unique hotspot site IDs and hotspot cluster IDs</span>
<span id="cb2-403"><a href="#cb2-403" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Explor alternatives to making hotspot clusters </span>
<span id="cb2-404"><a href="#cb2-404" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Name for the master data</span>
<span id="cb2-405"><a href="#cb2-405" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Need to add attributes to hotspot clusters (script 4)</span>
<span id="cb2-406"><a href="#cb2-406" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Mapping of hotspot cluster ID to hotspot IDs</span>
<span id="cb2-407"><a href="#cb2-407" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-408"><a href="#cb2-408" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-409"><a href="#cb2-409" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-410"><a href="#cb2-410" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-411"><a href="#cb2-411" aria-hidden="true" tabindex="-1"></a>::: {.callout-note collapse=&quot;true&quot;}</span>
<span id="cb2-412"><a href="#cb2-412" aria-hidden="true" tabindex="-1"></a><span class="fu"># 5/2/2025</span></span>
<span id="cb2-413"><a href="#cb2-413" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-414"><a href="#cb2-414" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Raahil, Matt</span></span>
<span id="cb2-415"><a href="#cb2-415" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-416"><a href="#cb2-416" aria-hidden="true" tabindex="-1"></a><span class="fu">## Agenda</span></span>
<span id="cb2-417"><a href="#cb2-417" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Site attributes data progress</span>
<span id="cb2-418"><a href="#cb2-418" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>In progress</span>
<span id="cb2-419"><a href="#cb2-419" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>Individual attributes data progress</span>
<span id="cb2-420"><a href="#cb2-420" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>Coding sample selection progress</span>
<span id="cb2-421"><a href="#cb2-421" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Done, flexible to time period and years (e.g. quarterly for at least 8 years)</span>
<span id="cb2-422"><a href="#cb2-422" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-423"><a href="#cb2-423" aria-hidden="true" tabindex="-1"></a><span class="fu">## Site Attributes</span></span>
<span id="cb2-424"><a href="#cb2-424" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-425"><a href="#cb2-425" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Still gathering geospatial data</span>
<span id="cb2-426"><a href="#cb2-426" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span><span class="co">[</span><span class="ot">MODIS VCF forest cover</span><span class="co">](https://lpdaac.usgs.gov/products/mod44bv061/)</span></span>
<span id="cb2-427"><a href="#cb2-427" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span><span class="co">[</span><span class="ot">Copernicus ERA5-Land</span><span class="co">](https://cds.climate.copernicus.eu/datasets/reanalysis-era5-land?tab=download)</span></span>
<span id="cb2-428"><a href="#cb2-428" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span><span class="co">[</span><span class="ot">ecmwfr</span><span class="co">](https://bluegreen-labs.github.io/ecmwfr/)</span>, R API interface</span>
<span id="cb2-429"><a href="#cb2-429" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Run zonal statistics on buffer around hotspots</span>
<span id="cb2-430"><a href="#cb2-430" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>If data becomes to large to easily share, we can start using Globus and MSI&#39;s tier 2 data storage</span>
<span id="cb2-431"><a href="#cb2-431" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-432"><a href="#cb2-432" aria-hidden="true" tabindex="-1"></a><span class="fu">## Next tasks</span></span>
<span id="cb2-433"><a href="#cb2-433" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-434"><a href="#cb2-434" aria-hidden="true" tabindex="-1"></a><span class="ss">1. </span>Merge site and individual attributes to make master dataset</span>
<span id="cb2-435"><a href="#cb2-435" aria-hidden="true" tabindex="-1"></a><span class="ss">2. </span>Calculate travel cost</span>
<span id="cb2-436"><a href="#cb2-436" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Start with linear distance for now</span>
<span id="cb2-437"><a href="#cb2-437" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Calculate cost</span>
<span id="cb2-438"><a href="#cb2-438" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Refer to Lloyd-Smith and Cameron/Kolstoe</span>
<span id="cb2-439"><a href="#cb2-439" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Travel cost = 2 (round trip) x distance x 1/3 x wage (from GDP)</span>
<span id="cb2-440"><a href="#cb2-440" aria-hidden="true" tabindex="-1"></a><span class="ss">3. </span>Species Richness</span>
<span id="cb2-441"><a href="#cb2-441" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Decide on method: </span>
<span id="cb2-442"><a href="#cb2-442" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Weitzman index</span>
<span id="cb2-443"><a href="#cb2-443" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>iNaturalist</span>
<span id="cb2-444"><a href="#cb2-444" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Simpson index </span>
<span id="cb2-445"><a href="#cb2-445" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Shannon index</span>
<span id="cb2-446"><a href="#cb2-446" aria-hidden="true" tabindex="-1"></a><span class="ss">4. </span>Looking ahead</span>
<span id="cb2-447"><a href="#cb2-447" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Look at code for estimating mixed logit with big data</span>
<span id="cb2-448"><a href="#cb2-448" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span><span class="co">[</span><span class="ot">mixl</span><span class="co">](https://github.com/joemolloy/fast-mixed-mnl)</span>, C++ based R package</span>
<span id="cb2-449"><a href="#cb2-449" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span><span class="co">[</span><span class="ot">xlogit</span><span class="co">](https://xlogit.readthedocs.io/en/latest/index.html)</span>, GPU-Accelerated Python Package</span>
<span id="cb2-450"><a href="#cb2-450" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>How to properly weight the WTP for aggregation</span>
<span id="cb2-451"><a href="#cb2-451" aria-hidden="true" tabindex="-1"></a>    <span class="co">&lt;!-- - [Weighting the Evidence: A Rank-Dependent Model of Outdoor Recreation](https://doi-org.ezp1.lib.umn.edu/10.1086/732185) --&gt;</span></span>
<span id="cb2-452"><a href="#cb2-452" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Research how to include fixed effects in utility functions (alternative specific constant)</span>
<span id="cb2-453"><a href="#cb2-453" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>User fixed effects </span>
<span id="cb2-454"><a href="#cb2-454" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>hotspot fixed effects</span>
<span id="cb2-455"><a href="#cb2-455" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-456"><a href="#cb2-456" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-457"><a href="#cb2-457" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-458"><a href="#cb2-458" aria-hidden="true" tabindex="-1"></a>::: {.callout-note  collapsse=&quot;true&quot;}</span>
<span id="cb2-459"><a href="#cb2-459" aria-hidden="true" tabindex="-1"></a><span class="fu"># 4/25/2025</span></span>
<span id="cb2-460"><a href="#cb2-460" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Matt, Ryan</span></span>
<span id="cb2-461"><a href="#cb2-461" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-462"><a href="#cb2-462" aria-hidden="true" tabindex="-1"></a><span class="fu">## Tasks: </span></span>
<span id="cb2-463"><a href="#cb2-463" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Matt: Download data and share</span>
<span id="cb2-464"><a href="#cb2-464" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Ryan: Start writing pseudocode in the new scripts. </span>
<span id="cb2-465"><a href="#cb2-465" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Both: Play with ClimateR </span>
<span id="cb2-466"><a href="#cb2-466" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-467"><a href="#cb2-467" aria-hidden="true" tabindex="-1"></a><span class="fu">## Site Attributes</span></span>
<span id="cb2-468"><a href="#cb2-468" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Limitation: </span>
<span id="cb2-469"><a href="#cb2-469" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>It takes time to build GIS data. So a lot of the attributes are not updated for 2024/25</span>
<span id="cb2-470"><a href="#cb2-470" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>ClimateR</span>
<span id="cb2-471"><a href="#cb2-471" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Division of the attributes to collect</span>
<span id="cb2-472"><a href="#cb2-472" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Protected area (binary or distance) -- fuzzy borders leads to spillovers</span>
<span id="cb2-473"><a href="#cb2-473" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Matt</span>
<span id="cb2-474"><a href="#cb2-474" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>WDPA (https://www.protectedplanet.net/en/thematic-areas/wdpa?tab=WDPA)</span>
<span id="cb2-475"><a href="#cb2-475" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Tree cover -- higher willingess to pay for forest</span>
<span id="cb2-476"><a href="#cb2-476" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Matt</span>
<span id="cb2-477"><a href="#cb2-477" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>https://data.fs.usda.gov/geodata/rastergateway/treecanopycover/</span>
<span id="cb2-478"><a href="#cb2-478" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>https://data.globalforestwatch.org/</span>
<span id="cb2-479"><a href="#cb2-479" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>https://glad.umd.edu/dataset/global-2010-tree-cover-30-m</span>
<span id="cb2-480"><a href="#cb2-480" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Worse comes to worse we use the ESA data</span>
<span id="cb2-481"><a href="#cb2-481" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Or we could use NDVI data as a proxy.</span>
<span id="cb2-482"><a href="#cb2-482" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Rainfall (monthly) - seasonality</span>
<span id="cb2-483"><a href="#cb2-483" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Ryan</span>
<span id="cb2-484"><a href="#cb2-484" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>https://ftp.cpc.ncep.noaa.gov/GIS/GRADS_GIS/GeoTIFF/GLB_DLY_PREC/</span>
<span id="cb2-485"><a href="#cb2-485" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>CHRIPS SPI data</span>
<span id="cb2-486"><a href="#cb2-486" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>https://mikejohnson51.github.io/climateR/index.html</span>
<span id="cb2-487"><a href="#cb2-487" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Temperature (monthly) - seasonality</span>
<span id="cb2-488"><a href="#cb2-488" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Ryan</span>
<span id="cb2-489"><a href="#cb2-489" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>GISS Surface Temperature Analysis (GISTEMP v3)</span>
<span id="cb2-490"><a href="#cb2-490" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Surface air temperature (no ocean data), 250km smoothing (9 MB)</span>
<span id="cb2-491"><a href="#cb2-491" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Might need the land mask</span>
<span id="cb2-492"><a href="#cb2-492" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>https://data.giss.nasa.gov/gistemp/</span>
<span id="cb2-493"><a href="#cb2-493" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Distance to the coast -- species gradient to the coast</span>
<span id="cb2-494"><a href="#cb2-494" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Ryan</span>
<span id="cb2-495"><a href="#cb2-495" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Dissolve the districts data. </span>
<span id="cb2-496"><a href="#cb2-496" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Make a polyline and measure distance to the nearest line</span>
<span id="cb2-497"><a href="#cb2-497" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Species richness from the previous week (expectation) -- and any other measures of biodiversity</span>
<span id="cb2-498"><a href="#cb2-498" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Both Matt and Ryan</span>
<span id="cb2-499"><a href="#cb2-499" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Weitzmen index using ebird data to generate diversity score</span>
<span id="cb2-500"><a href="#cb2-500" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>https://scholar.harvard.edu/files/weitzman/files/on_diversity.pdf</span>
<span id="cb2-501"><a href="#cb2-501" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Validate with inaturalist data on birds for india</span>
<span id="cb2-502"><a href="#cb2-502" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Could test a couple of different indices</span>
<span id="cb2-503"><a href="#cb2-503" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>How we are sharing the site attribute data</span>
<span id="cb2-504"><a href="#cb2-504" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Matt will have the GDrive link. </span>
<span id="cb2-505"><a href="#cb2-505" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Updating the config files </span>
<span id="cb2-506"><a href="#cb2-506" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Once we have data download</span>
<span id="cb2-507"><a href="#cb2-507" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Start script to merge site attributes to sites</span>
<span id="cb2-508"><a href="#cb2-508" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>What to do about 4.demographics. Seems like that should be site attributes. </span>
<span id="cb2-509"><a href="#cb2-509" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Noteably, some of the data will need to become panel. </span>
<span id="cb2-510"><a href="#cb2-510" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Rainfall</span>
<span id="cb2-511"><a href="#cb2-511" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Temperature</span>
<span id="cb2-512"><a href="#cb2-512" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Protected Areas</span>
<span id="cb2-513"><a href="#cb2-513" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Tree cover</span>
<span id="cb2-514"><a href="#cb2-514" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>species richness (we need to dig some that has a time dimension)</span>
<span id="cb2-515"><a href="#cb2-515" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Static: </span>
<span id="cb2-516"><a href="#cb2-516" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Distance to coast</span>
<span id="cb2-517"><a href="#cb2-517" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-518"><a href="#cb2-518" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-519"><a href="#cb2-519" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-520"><a href="#cb2-520" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-521"><a href="#cb2-521" aria-hidden="true" tabindex="-1"></a>::: {.callout-note  collapse=&quot;true&quot;}</span>
<span id="cb2-522"><a href="#cb2-522" aria-hidden="true" tabindex="-1"></a><span class="fu"># 4/18/2025</span></span>
<span id="cb2-523"><a href="#cb2-523" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Raahil, Matt, Jovin, Ryan</span></span>
<span id="cb2-524"><a href="#cb2-524" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-525"><a href="#cb2-525" aria-hidden="true" tabindex="-1"></a><span class="fu">## Sample Selection Updates</span></span>
<span id="cb2-526"><a href="#cb2-526" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>different biodiversity levels by incorporating interaction terms in locals and tourists. WTP for nature tourism vs WTP for nature recreation or biodiverstiy</span>
<span id="cb2-527"><a href="#cb2-527" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>If we estimate both, then we can compare both measures. We can also place it within the larger literature of many WTP for both subsets of ecosystem services and the population using those services. </span>
<span id="cb2-528"><a href="#cb2-528" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Broadly compare conservation, nature recreation, tourism, or biodiversity </span>
<span id="cb2-529"><a href="#cb2-529" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>We need to frame this as a development paper as the lambda in the Envirodevonomics paper</span>
<span id="cb2-530"><a href="#cb2-530" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Frequency of observations are consistent over time and potentially visits near protected areas. </span>
<span id="cb2-531"><a href="#cb2-531" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Need to capture seasonanlity (time fixed effects)</span>
<span id="cb2-532"><a href="#cb2-532" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Need to consider some aspect of congestion (can calculate that as a site attribute)</span>
<span id="cb2-533"><a href="#cb2-533" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>If you go two months a year you are less likely to be a tourist</span>
<span id="cb2-534"><a href="#cb2-534" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-535"><a href="#cb2-535" aria-hidden="true" tabindex="-1"></a><span class="fu">## Literature Review Updates</span></span>
<span id="cb2-536"><a href="#cb2-536" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We can uses species richness as an explanatory variable in the model. But could add other biodiversity measures beyond bird diversity. </span>
<span id="cb2-537"><a href="#cb2-537" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Need to think about how to frame this in the developing context instead of the environment context</span>
<span id="cb2-538"><a href="#cb2-538" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Very few WTP for environmental quality in development economics space </span>
<span id="cb2-539"><a href="#cb2-539" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Would be great if our WTP measure could be placed within a welfare model so we can make counterfactual statements</span>
<span id="cb2-540"><a href="#cb2-540" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Need to have homogenous agents to aggregate for welfare gains &gt;&gt; e-bird users are not representative of population</span>
<span id="cb2-541"><a href="#cb2-541" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Could overcome this by population demographic weights to make e-bird users representative</span>
<span id="cb2-542"><a href="#cb2-542" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>But welfare statements are a big improvement over the other e-bird papers</span>
<span id="cb2-543"><a href="#cb2-543" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Could explore the WTP measure of space and time. So we go beyond the cross-sectional estimates in the literature </span>
<span id="cb2-544"><a href="#cb2-544" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>If we have 1st and 2nd moment, then we can assign individuals an individual WTP. </span>
<span id="cb2-545"><a href="#cb2-545" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-546"><a href="#cb2-546" aria-hidden="true" tabindex="-1"></a><span class="fu">## Grant Proposal</span></span>
<span id="cb2-547"><a href="#cb2-547" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Jovin will provide an update. </span>
<span id="cb2-548"><a href="#cb2-548" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-549"><a href="#cb2-549" aria-hidden="true" tabindex="-1"></a><span class="fu">## Initial Results by Raahil </span></span>
<span id="cb2-550"><a href="#cb2-550" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We are getting coverage of the country </span>
<span id="cb2-551"><a href="#cb2-551" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Some overlap of population density and users &gt;&gt; likely representative</span>
<span id="cb2-552"><a href="#cb2-552" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Most hotspots and users are in the south (consistent with e-birder users)</span>
<span id="cb2-553"><a href="#cb2-553" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Hotspots data could be weighted by visits</span>
<span id="cb2-554"><a href="#cb2-554" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Should plot protected areas overlapping hotspots</span>
<span id="cb2-555"><a href="#cb2-555" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Demographic t-test shows e-bird users are richer on average. But we can use this to back out our inverse weight for the welfare calculations</span>
<span id="cb2-556"><a href="#cb2-556" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>E-bird users are selecting into this where there are richer places (so can afford this hobby)</span>
<span id="cb2-557"><a href="#cb2-557" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-558"><a href="#cb2-558" aria-hidden="true" tabindex="-1"></a><span class="fu">## Tasks</span></span>
<span id="cb2-559"><a href="#cb2-559" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>How do we distinguish ourselves from the Pacific Northwest paper Kolestet 2017</span>
<span id="cb2-560"><a href="#cb2-560" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>First pass of the sample: users who use this at least 2 months out of the year</span>
<span id="cb2-561"><a href="#cb2-561" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Make two new scripts: </span>
<span id="cb2-562"><a href="#cb2-562" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Merging in site attributes (Can merge in satilette data -- radius around the centriod and take the mean -- and district data about the site attribtues -- merge on district ID)</span>
<span id="cb2-563"><a href="#cb2-563" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Protected area (binary or distance) -- fuzzy borders leads to spillovers</span>
<span id="cb2-564"><a href="#cb2-564" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Tree cover -- higher willingess to pay for forest</span>
<span id="cb2-565"><a href="#cb2-565" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Rainfall (monthly) - seasonality</span>
<span id="cb2-566"><a href="#cb2-566" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Temperature (monthly) - seasonality</span>
<span id="cb2-567"><a href="#cb2-567" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Distance to the coast -- species gradient to the coast</span>
<span id="cb2-568"><a href="#cb2-568" aria-hidden="true" tabindex="-1"></a><span class="ss">        - </span>Species richness from the previous week (expectation) -- and any other measures of biodiversity</span>
<span id="cb2-569"><a href="#cb2-569" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Estimating the RUM via the mixed multinominal logit</span>
<span id="cb2-570"><a href="#cb2-570" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Email tag for grant proposals</span>
<span id="cb2-571"><a href="#cb2-571" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Make a script into a single PDF</span>
<span id="cb2-572"><a href="#cb2-572" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-573"><a href="#cb2-573" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-574"><a href="#cb2-574" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-575"><a href="#cb2-575" aria-hidden="true" tabindex="-1"></a>::: {.callout-note  collapse=&quot;true&quot;}</span>
<span id="cb2-576"><a href="#cb2-576" aria-hidden="true" tabindex="-1"></a><span class="fu"># 3/31/2025</span></span>
<span id="cb2-577"><a href="#cb2-577" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Raahil, Matt, Jovin</span></span>
<span id="cb2-578"><a href="#cb2-578" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-579"><a href="#cb2-579" aria-hidden="true" tabindex="-1"></a><span class="fu">## Sample Selection Discussion</span></span>
<span id="cb2-580"><a href="#cb2-580" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We want to ensure that the sample of ebird observers is representative</span>
<span id="cb2-581"><a href="#cb2-581" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Remove tourists</span>
<span id="cb2-582"><a href="#cb2-582" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Remove infrequent users</span>
<span id="cb2-583"><a href="#cb2-583" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Potential strategies </span>
<span id="cb2-584"><a href="#cb2-584" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Limit to users that log at least a certain number of times per month</span>
<span id="cb2-585"><a href="#cb2-585" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Exclude users that post outside of protected areas</span>
<span id="cb2-586"><a href="#cb2-586" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Exclude users that include tourism tags in text variable(e.g. &quot;ecolodge&quot;)</span>
<span id="cb2-587"><a href="#cb2-587" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-588"><a href="#cb2-588" aria-hidden="true" tabindex="-1"></a><span class="fu">## Tasks</span></span>
<span id="cb2-589"><a href="#cb2-589" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Matt/Jovin: descriptive analysis on data to determine distributions of frequent users</span>
<span id="cb2-590"><a href="#cb2-590" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Complete by 4/18</span>
<span id="cb2-591"><a href="#cb2-591" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-592"><a href="#cb2-592" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-593"><a href="#cb2-593" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-594"><a href="#cb2-594" aria-hidden="true" tabindex="-1"></a>::: {.callout-note  collapse=&quot;true&quot;}</span>
<span id="cb2-595"><a href="#cb2-595" aria-hidden="true" tabindex="-1"></a><span class="fu"># 3/21/2025</span></span>
<span id="cb2-596"><a href="#cb2-596" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Raahil, Matt, Ryan, Jovin</span></span>
<span id="cb2-597"><a href="#cb2-597" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-598"><a href="#cb2-598" aria-hidden="true" tabindex="-1"></a><span class="fu">## Action Items: </span></span>
<span id="cb2-599"><a href="#cb2-599" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-600"><a href="#cb2-600" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Everyone confirms that we can run the first 3 basic scripts</span>
<span id="cb2-601"><a href="#cb2-601" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Starting the annotated bibliography with 10 most important papers: </span>
<span id="cb2-602"><a href="#cb2-602" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>WTP for envrionmental quality </span>
<span id="cb2-603"><a href="#cb2-603" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>WTP in developing countries using travel cost</span>
<span id="cb2-604"><a href="#cb2-604" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Review of mixed logit methodology</span>
<span id="cb2-605"><a href="#cb2-605" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Next week we need to talk about sample selection in depth next week </span>
<span id="cb2-606"><a href="#cb2-606" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-607"><a href="#cb2-607" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-608"><a href="#cb2-608" aria-hidden="true" tabindex="-1"></a><span class="fu"># Division of Tasks: </span></span>
<span id="cb2-609"><a href="#cb2-609" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Review mixed logit method</span>
<span id="cb2-610"><a href="#cb2-610" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>(Matt) Review of WTP in developing countries using travel cost methods</span>
<span id="cb2-611"><a href="#cb2-611" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>(Ryan) Review of WTP methods for environmental quality</span>
<span id="cb2-612"><a href="#cb2-612" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>(Ryan) Method to estimate welfare effects (some exogenous shock)</span>
<span id="cb2-613"><a href="#cb2-613" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>(Raahil + Jovin + Matt) Determining the sample population criteria -- important early step</span>
<span id="cb2-614"><a href="#cb2-614" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>(Raahil) Gathering data for population demographics (SHRUG, DHS, etc.)</span>
<span id="cb2-615"><a href="#cb2-615" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-616"><a href="#cb2-616" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-617"><a href="#cb2-617" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Ways to annoate bibliography: </span>
<span id="cb2-618"><a href="#cb2-618" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>paper title + journal,  research question, research design (e.g. stated preference), data/sample size, result (i.e. WTP estimate)</span>
<span id="cb2-619"><a href="#cb2-619" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We need to think about how we contribute. </span>
<span id="cb2-620"><a href="#cb2-620" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We need to think about sample selection. </span>
<span id="cb2-621"><a href="#cb2-621" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Criteria based on usage, overlap with protected areas, etc. </span>
<span id="cb2-622"><a href="#cb2-622" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>How do we remove tourists? </span>
<span id="cb2-623"><a href="#cb2-623" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Who is the average person? Estimates will be sensitive</span>
<span id="cb2-624"><a href="#cb2-624" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Need to gridded map of ebird observations</span>
<span id="cb2-625"><a href="#cb2-625" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-626"><a href="#cb2-626" aria-hidden="true" tabindex="-1"></a><span class="fu"># Next scripts</span></span>
<span id="cb2-627"><a href="#cb2-627" aria-hidden="true" tabindex="-1"></a><span class="ss">4. </span>Merge attributes of the counterfactual sites. Need to collect site level data and/or individual level data </span>
<span id="cb2-628"><a href="#cb2-628" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Site attributes: biodiversity, accessibility, ruggedness, congestion, temperature, rainfall, regional developmemnt</span>
<span id="cb2-629"><a href="#cb2-629" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>For first run, just do biodiversity. </span>
<span id="cb2-630"><a href="#cb2-630" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Biodiversity comes from either diversity of birds or other attributes of ecosystem diversity</span>
<span id="cb2-631"><a href="#cb2-631" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Expected biodiversity is why you go. So we can use historical measures of biodiversity</span>
<span id="cb2-632"><a href="#cb2-632" aria-hidden="true" tabindex="-1"></a><span class="ss">5. </span>Run the mixed logit to estimate the WTP values</span>
<span id="cb2-633"><a href="#cb2-633" aria-hidden="true" tabindex="-1"></a><span class="ss">6. </span>Welfare analysis of WTP values</span>
<span id="cb2-634"><a href="#cb2-634" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-635"><a href="#cb2-635" aria-hidden="true" tabindex="-1"></a>:::</span>
<span id="cb2-636"><a href="#cb2-636" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-637"><a href="#cb2-637" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-638"><a href="#cb2-638" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-639"><a href="#cb2-639" aria-hidden="true" tabindex="-1"></a>::: {.callout-note  collapse=&quot;true&quot;}</span>
<span id="cb2-640"><a href="#cb2-640" aria-hidden="true" tabindex="-1"></a><span class="fu"># 02/28/25</span></span>
<span id="cb2-641"><a href="#cb2-641" aria-hidden="true" tabindex="-1"></a><span class="at">&gt; Raahil, Matt, Ryan</span></span>
<span id="cb2-642"><a href="#cb2-642" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-643"><a href="#cb2-643" aria-hidden="true" tabindex="-1"></a><span class="fu">## Action Items: </span></span>
<span id="cb2-644"><a href="#cb2-644" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-645"><a href="#cb2-645" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Ryan: Get the correct e-bird file from Matt (DONE)</span>
<span id="cb2-646"><a href="#cb2-646" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Ryan: Merge code files into scripts folder. (DONE)</span>
<span id="cb2-647"><a href="#cb2-647" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Ryan: Initate a document for literature review with template example (DONE)</span>
<span id="cb2-648"><a href="#cb2-648" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We need to specalize and define tasks. Volunteer over email. Tasks: Lit review, sample selection, characterize the sample, theory person (DONE)</span>
<span id="cb2-649"><a href="#cb2-649" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>And set up a recurring meeting time. Need a poll. Try for next two weeks from now. (DONE)</span>
<span id="cb2-650"><a href="#cb2-650" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Jovin: Provide your github username so Ryan can make you a collaborator on the github repo.  (DONE)</span>
<span id="cb2-651"><a href="#cb2-651" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-652"><a href="#cb2-652" aria-hidden="true" tabindex="-1"></a><span class="fu">## Funding: </span></span>
<span id="cb2-653"><a href="#cb2-653" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-654"><a href="#cb2-654" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>No longer pursue IonE funding</span>
<span id="cb2-655"><a href="#cb2-655" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-656"><a href="#cb2-656" aria-hidden="true" tabindex="-1"></a><span class="fu">## Coding: </span></span>
<span id="cb2-657"><a href="#cb2-657" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-658"><a href="#cb2-658" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Matt was able to replicate everything </span>
<span id="cb2-659"><a href="#cb2-659" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Created a YAML to process file names</span>
<span id="cb2-660"><a href="#cb2-660" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>No longer needs new file paths</span>
<span id="cb2-661"><a href="#cb2-661" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Ryan stuck on loading data</span>
<span id="cb2-662"><a href="#cb2-662" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Need to all use the same exact file download for e-bird</span>
<span id="cb2-663"><a href="#cb2-663" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>script 1: processes e-bird data, determines user homes, and calculates home for those who don&#39;t report it. </span>
<span id="cb2-664"><a href="#cb2-664" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>script 2: Creates the hotspot observations. Measures distance from home to hotspot.</span>
<span id="cb2-665"><a href="#cb2-665" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>script 3: Most important. 12,000 hotspots so too many counterfactuals. This script reduces the counterfactual set by clustering hotspots nearby (likley parks) and only consider hotspots within a distance of home. </span>
<span id="cb2-666"><a href="#cb2-666" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>The final data set is a panel of person (user) ID by trip ID appended to all counterfactual trips for this user. </span>
<span id="cb2-667"><a href="#cb2-667" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>When we actually estimate this, we will need to plan for a no-trip counterfactual. </span>
<span id="cb2-668"><a href="#cb2-668" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>One of the main things we need to do is make the computation easier for us. Otherwise it becomes too computationally intensive. </span>
<span id="cb2-669"><a href="#cb2-669" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Keep all the scripts in scripts folder. </span>
<span id="cb2-670"><a href="#cb2-670" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-671"><a href="#cb2-671" aria-hidden="true" tabindex="-1"></a><span class="fu">## Data: </span></span>
<span id="cb2-672"><a href="#cb2-672" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>We will need to bring in site attribute data </span>
<span id="cb2-673"><a href="#cb2-673" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>So data on landscape measurements. </span>
<span id="cb2-674"><a href="#cb2-674" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Matt says that not all intermediate files are being save</span>
<span id="cb2-675"><a href="#cb2-675" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Specifically, the master file is not saving</span>
<span id="cb2-676"><a href="#cb2-676" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Need to identify our sample population</span>
<span id="cb2-677"><a href="#cb2-677" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Need a careful (defensiable) data-driven criteria for sample </span>
<span id="cb2-678"><a href="#cb2-678" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Needs to be regular, local users (not temporary or tourists)</span>
<span id="cb2-679"><a href="#cb2-679" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Determine this based on usage frequency and mobility patterns</span>
<span id="cb2-680"><a href="#cb2-680" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Consider overlaying protected areas polygons on top of the observations/hotspots</span>
<span id="cb2-681"><a href="#cb2-681" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Buffers may create a censoring problem. So we would just test this against larger and larger buffers</span>
<span id="cb2-682"><a href="#cb2-682" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Need to find ways to characteristize the users (demographics) without having any information about them. </span>
<span id="cb2-683"><a href="#cb2-683" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Important for welfare calculations. </span>
<span id="cb2-684"><a href="#cb2-684" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Try to match with DHS data </span>
<span id="cb2-685"><a href="#cb2-685" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Or overlap home location with SHRUG data set</span>
<span id="cb2-686"><a href="#cb2-686" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Econometrically, take attribtues about where they live against the average for the country, how do high and low WTP birders fall within the distribution</span>
<span id="cb2-687"><a href="#cb2-687" aria-hidden="true" tabindex="-1"></a><span class="ss">    - </span>Potentially use this as weights for welfare calculations. </span>
<span id="cb2-688"><a href="#cb2-688" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-689"><a href="#cb2-689" aria-hidden="true" tabindex="-1"></a><span class="fu">## Methods:</span></span>
<span id="cb2-690"><a href="#cb2-690" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>If we get a random shock to bird population, then we can think about the causal effect of increasing biodiversity</span>
<span id="cb2-691"><a href="#cb2-691" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-692"><a href="#cb2-692" aria-hidden="true" tabindex="-1"></a><span class="fu">## Paper/Writing: </span></span>
<span id="cb2-693"><a href="#cb2-693" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>For the ML cluster, will want to read up later to find out if there is a better way or more common practice in economics. </span>
<span id="cb2-694"><a href="#cb2-694" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Create a lit review. Three groups: </span>
<span id="cb2-695"><a href="#cb2-695" aria-hidden="true" tabindex="-1"></a><span class="ss">    1. </span>estimates of WTP in developing countries generally using the discrete choice travel cost method (Ted Miguel water quality paper). </span>
<span id="cb2-696"><a href="#cb2-696" aria-hidden="true" tabindex="-1"></a><span class="ss">    2. </span>WTP for environmental quality papers and what design they are using to estimate (e.g., contingent valuation, stated preferences -- something from ecology)</span>
<span id="cb2-697"><a href="#cb2-697" aria-hidden="true" tabindex="-1"></a><span class="ss">    3. </span>Review of any updated versions of the mixed logit model -- new adapations of the model from class</span>
<span id="cb2-698"><a href="#cb2-698" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>For this: question, data used, design, and finding (dollar estimates would be great).</span>
<span id="cb2-699"><a href="#cb2-699" aria-hidden="true" tabindex="-1"></a><span class="ss">- </span>Create a document in the docs folder</span>
<span id="cb2-700"><a href="#cb2-700" aria-hidden="true" tabindex="-1"></a>:::</span></code></pre></div>
</div>
</section>

</main>
<!-- /main column -->
<script id = "quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const isCodeAnnotation = (el) => {
    for (const clz of el.classList) {
      if (clz.startsWith('code-annotation-')) {                     
        return true;
      }
    }
    return false;
  }
  const onCopySuccess = function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    let tooltip;
    if (window.bootstrap) {
      button.setAttribute("data-bs-toggle", "tooltip");
      button.setAttribute("data-bs-placement", "left");
      button.setAttribute("data-bs-title", "Copied!");
      tooltip = new bootstrap.Tooltip(button, 
        { trigger: "manual", 
          customClass: "code-copy-button-tooltip",
          offset: [0, -8]});
      tooltip.show();    
    }
    setTimeout(function() {
      if (tooltip) {
        tooltip.hide();
        button.removeAttribute("data-bs-title");
        button.removeAttribute("data-bs-toggle");
        button.removeAttribute("data-bs-placement");
      }
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  }
  const getTextToCopy = function(trigger) {
      const codeEl = trigger.previousElementSibling.cloneNode(true);
      for (const childEl of codeEl.children) {
        if (isCodeAnnotation(childEl)) {
          childEl.remove();
        }
      }
      return codeEl.innerText;
  }
  const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
    text: getTextToCopy
  });
  clipboard.on('success', onCopySuccess);
  if (window.document.getElementById('quarto-embedded-source-code-modal')) {
    const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
      text: getTextToCopy,
      container: window.document.getElementById('quarto-embedded-source-code-modal')
    });
    clipboardModal.on('success', onCopySuccess);
  }
  const viewSource = window.document.getElementById('quarto-view-source') ||
                     window.document.getElementById('quarto-code-tools-source');
  if (viewSource) {
    const sourceUrl = viewSource.getAttribute("data-quarto-source-url");
    viewSource.addEventListener("click", function(e) {
      if (sourceUrl) {
        // rstudio viewer pane
        if (/\bcapabilities=\b/.test(window.location)) {
          window.open(sourceUrl);
        } else {
          window.location.href = sourceUrl;
        }
      } else {
        const modal = new bootstrap.Modal(document.getElementById('quarto-embedded-source-code-modal'));
        modal.show();
      }
      return false;
    });
  }
  function toggleCodeHandler(show) {
    return function(e) {
      const detailsSrc = window.document.querySelectorAll(".cell > details > .sourceCode");
      for (let i=0; i<detailsSrc.length; i++) {
        const details = detailsSrc[i].parentElement;
        if (show) {
          details.open = true;
        } else {
          details.removeAttribute("open");
        }
      }
      const cellCodeDivs = window.document.querySelectorAll(".cell > .sourceCode");
      const fromCls = show ? "hidden" : "unhidden";
      const toCls = show ? "unhidden" : "hidden";
      for (let i=0; i<cellCodeDivs.length; i++) {
        const codeDiv = cellCodeDivs[i];
        if (codeDiv.classList.contains(fromCls)) {
          codeDiv.classList.remove(fromCls);
          codeDiv.classList.add(toCls);
        } 
      }
      return false;
    }
  }
  const hideAllCode = window.document.getElementById("quarto-hide-all-code");
  if (hideAllCode) {
    hideAllCode.addEventListener("click", toggleCodeHandler(false));
  }
  const showAllCode = window.document.getElementById("quarto-show-all-code");
  if (showAllCode) {
    showAllCode.addEventListener("click", toggleCodeHandler(true));
  }
    var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
    var mailtoRegex = new RegExp(/^mailto:/);
      var filterRegex = new RegExp('/' + window.location.host + '/');
    var isInternal = (href) => {
        return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
    }
    // Inspect non-navigation links and adorn them if external
 	var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
    for (var i=0; i<links.length; i++) {
      const link = links[i];
      if (!isInternal(link.href)) {
        // undo the damage that might have been done by quarto-nav.js in the case of
        // links that we want to consider external
        if (link.dataset.originalHref !== undefined) {
          link.href = link.dataset.originalHref;
        }
      }
    }
  function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
    const config = {
      allowHTML: true,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start',
    };
    if (contentFn) {
      config.content = contentFn;
    }
    if (onTriggerFn) {
      config.onTrigger = onTriggerFn;
    }
    if (onUntriggerFn) {
      config.onUntrigger = onUntriggerFn;
    }
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      if (note) {
        return note.innerHTML;
      } else {
        return "";
      }
    });
  }
  const xrefs = window.document.querySelectorAll('a.quarto-xref');
  const processXRef = (id, note) => {
    // Strip column container classes
    const stripColumnClz = (el) => {
      el.classList.remove("page-full", "page-columns");
      if (el.children) {
        for (const child of el.children) {
          stripColumnClz(child);
        }
      }
    }
    stripColumnClz(note)
    if (id === null || id.startsWith('sec-')) {
      // Special case sections, only their first couple elements
      const container = document.createElement("div");
      if (note.children && note.children.length > 2) {
        container.appendChild(note.children[0].cloneNode(true));
        for (let i = 1; i < note.children.length; i++) {
          const child = note.children[i];
          if (child.tagName === "P" && child.innerText === "") {
            continue;
          } else {
            container.appendChild(child.cloneNode(true));
            break;
          }
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(container);
        }
        return container.innerHTML
      } else {
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        return note.innerHTML;
      }
    } else {
      // Remove any anchor links if they are present
      const anchorLink = note.querySelector('a.anchorjs-link');
      if (anchorLink) {
        anchorLink.remove();
      }
      if (window.Quarto?.typesetMath) {
        window.Quarto.typesetMath(note);
      }
      if (note.classList.contains("callout")) {
        return note.outerHTML;
      } else {
        return note.innerHTML;
      }
    }
  }
  for (var i=0; i<xrefs.length; i++) {
    const xref = xrefs[i];
    tippyHover(xref, undefined, function(instance) {
      instance.disable();
      let url = xref.getAttribute('href');
      let hash = undefined; 
      if (url.startsWith('#')) {
        hash = url;
      } else {
        try { hash = new URL(url).hash; } catch {}
      }
      if (hash) {
        const id = hash.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note !== null) {
          try {
            const html = processXRef(id, note.cloneNode(true));
            instance.setContent(html);
          } finally {
            instance.enable();
            instance.show();
          }
        } else {
          // See if we can fetch this
          fetch(url.split('#')[0])
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.getElementById(id);
            if (note !== null) {
              const html = processXRef(id, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      } else {
        // See if we can fetch a full url (with no hash to target)
        // This is a special case and we should probably do some content thinning / targeting
        fetch(url)
        .then(res => res.text())
        .then(html => {
          const parser = new DOMParser();
          const htmlDoc = parser.parseFromString(html, "text/html");
          const note = htmlDoc.querySelector('main.content');
          if (note !== null) {
            // This should only happen for chapter cross references
            // (since there is no id in the URL)
            // remove the first header
            if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
              note.children[0].remove();
            }
            const html = processXRef(null, note);
            instance.setContent(html);
          } 
        }).finally(() => {
          instance.enable();
          instance.show();
        });
      }
    }, function(instance) {
    });
  }
      let selectedAnnoteEl;
      const selectorForAnnotation = ( cell, annotation) => {
        let cellAttr = 'data-code-cell="' + cell + '"';
        let lineAttr = 'data-code-annotation="' +  annotation + '"';
        const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
        return selector;
      }
      const selectCodeLines = (annoteEl) => {
        const doc = window.document;
        const targetCell = annoteEl.getAttribute("data-target-cell");
        const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
        const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
        const lines = annoteSpan.getAttribute("data-code-lines").split(",");
        const lineIds = lines.map((line) => {
          return targetCell + "-" + line;
        })
        let top = null;
        let height = null;
        let parent = null;
        if (lineIds.length > 0) {
            //compute the position of the single el (top and bottom and make a div)
            const el = window.document.getElementById(lineIds[0]);
            top = el.offsetTop;
            height = el.offsetHeight;
            parent = el.parentElement.parentElement;
          if (lineIds.length > 1) {
            const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
            const bottom = lastEl.offsetTop + lastEl.offsetHeight;
            height = bottom - top;
          }
          if (top !== null && height !== null && parent !== null) {
            // cook up a div (if necessary) and position it 
            let div = window.document.getElementById("code-annotation-line-highlight");
            if (div === null) {
              div = window.document.createElement("div");
              div.setAttribute("id", "code-annotation-line-highlight");
              div.style.position = 'absolute';
              parent.appendChild(div);
            }
            div.style.top = top - 2 + "px";
            div.style.height = height + 4 + "px";
            div.style.left = 0;
            let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
            if (gutterDiv === null) {
              gutterDiv = window.document.createElement("div");
              gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
              gutterDiv.style.position = 'absolute';
              const codeCell = window.document.getElementById(targetCell);
              const gutter = codeCell.querySelector('.code-annotation-gutter');
              gutter.appendChild(gutterDiv);
            }
            gutterDiv.style.top = top - 2 + "px";
            gutterDiv.style.height = height + 4 + "px";
          }
          selectedAnnoteEl = annoteEl;
        }
      };
      const unselectCodeLines = () => {
        const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
        elementsIds.forEach((elId) => {
          const div = window.document.getElementById(elId);
          if (div) {
            div.remove();
          }
        });
        selectedAnnoteEl = undefined;
      };
        // Handle positioning of the toggle
    window.addEventListener(
      "resize",
      throttle(() => {
        elRect = undefined;
        if (selectedAnnoteEl) {
          selectCodeLines(selectedAnnoteEl);
        }
      }, 10)
    );
    function throttle(fn, ms) {
    let throttle = false;
    let timer;
      return (...args) => {
        if(!throttle) { // first call gets through
            fn.apply(this, args);
            throttle = true;
        } else { // all the others get throttled
            if(timer) clearTimeout(timer); // cancel #2
            timer = setTimeout(() => {
              fn.apply(this, args);
              timer = throttle = false;
            }, ms);
        }
      };
    }
      // Attach click handler to the DT
      const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
      for (const annoteDlNode of annoteDls) {
        annoteDlNode.addEventListener('click', (event) => {
          const clickedEl = event.target;
          if (clickedEl !== selectedAnnoteEl) {
            unselectCodeLines();
            const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
            if (activeEl) {
              activeEl.classList.remove('code-annotation-active');
            }
            selectCodeLines(clickedEl);
            clickedEl.classList.add('code-annotation-active');
          } else {
            // Unselect the line
            unselectCodeLines();
            clickedEl.classList.remove('code-annotation-active');
          }
        });
      }
  const findCites = (el) => {
    const parentEl = el.parentElement;
    if (parentEl) {
      const cites = parentEl.dataset.cites;
      if (cites) {
        return {
          el,
          cites: cites.split(' ')
        };
      } else {
        return findCites(el.parentElement)
      }
    } else {
      return undefined;
    }
  };
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const citeInfo = findCites(ref);
    if (citeInfo) {
      tippyHover(citeInfo.el, function() {
        var popup = window.document.createElement('div');
        citeInfo.cites.forEach(function(cite) {
          var citeDiv = window.document.createElement('div');
          citeDiv.classList.add('hanging-indent');
          citeDiv.classList.add('csl-entry');
          var biblioDiv = window.document.getElementById('ref-' + cite);
          if (biblioDiv) {
            citeDiv.innerHTML = biblioDiv.innerHTML;
          }
          popup.appendChild(citeDiv);
        });
        return popup.innerHTML;
      });
    }
  }
});
</script>
</div> <!-- /content -->

</body>

</html>
